<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Jason Willwerscheid" />


<title>Montoro et al. structure plots</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/core-js-2.5.3/shim.min.js"></script>
<script src="site_libs/react-18.2.0/react.min.js"></script>
<script src="site_libs/react-18.2.0/react-dom.min.js"></script>
<script src="site_libs/reactwidget-2.0.0/react-tools.js"></script>
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/reactable-0.4.4/reactable.css" rel="stylesheet" />
<script src="site_libs/reactable-binding-0.4.4/reactable.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">ebmf-for-scrnaseq-paper</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">Montoro et al. structure plots</h1>
<h4 class="author">Jason Willwerscheid</h4>
<h4 class="date">2/2/2025</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-02-06
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ebmf-for-scrnaseq-paper/</code>
<span class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220420code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20220420)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220420code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220420)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomwillwerscheidebmfforscrnaseqpapertree949f54540fcadbce0a41b0ee9955befcdd6a4be2targetblank949f545a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/willwerscheid/ebmf-for-scrnaseq-paper/tree/949f54540fcadbce0a41b0ee9955befcdd6a4be2" target="_blank">949f545</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomwillwerscheidebmfforscrnaseqpapertree949f54540fcadbce0a41b0ee9955befcdd6a4be2targetblank949f545a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/willwerscheid/ebmf-for-scrnaseq-paper/tree/949f54540fcadbce0a41b0ee9955befcdd6a4be2" target="_blank">949f545</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/.DS_Store
    Ignored:    data/montoro-mean-expr.rds
    Ignored:    data/montoro.rds
    Ignored:    data/pijuan-sala-mean-expr.rds
    Ignored:    data/pijuan-sala.rds
    Ignored:    figs/
    Ignored:    output/montoro-ebmf.rds
    Ignored:    output/montoro-ebnmf-gsea.rds
    Ignored:    output/montoro-nmf-gsea.rds
    Ignored:    output/montoro-nmf.rds
    Ignored:    output/montoro-tm-gsea.rds
    Ignored:    output/montoro-topics.rds
    Ignored:    output/ps-ebmf-20.rds
    Ignored:    output/ps-ebmf-30.rds
    Ignored:    output/ps-ebmf-40.rds
    Ignored:    output/ps-nmf.rds
    Ignored:    output/ps-topics-20.rds
    Ignored:    output/ps-topics-30.rds
    Ignored:    output/ps-topics-40.rds

Untracked files:
    Untracked:  code/montoro_gseasusie2.R

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/montoro_structure.Rmd</code>) and
HTML (<code>docs/montoro_structure.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/willwerscheid/ebmf-for-scrnaseq-paper/blob/949f54540fcadbce0a41b0ee9955befcdd6a4be2/analysis/montoro_structure.Rmd" target="_blank">949f545</a>
</td>
<td>
Jason Willwerscheid
</td>
<td>
2025-02-06
</td>
<td>
workflowr::wflow_publish("analysis/montoro_structure.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/willwerscheid/ebmf-for-scrnaseq-paper/2e18b60ce2723f5c4a9b6d0add2c8afc4e5793cd/docs/montoro_structure.html" target="_blank">2e18b60</a>
</td>
<td>
Jason Willwerscheid
</td>
<td>
2025-02-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/willwerscheid/ebmf-for-scrnaseq-paper/blob/87d10f4264c45213eafecde52fc195a3312ed8df/analysis/montoro_structure.Rmd" target="_blank">87d10f4</a>
</td>
<td>
Jason Willwerscheid
</td>
<td>
2025-02-05
</td>
<td>
wflow_publish("analysis/montoro_structure.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/willwerscheid/ebmf-for-scrnaseq-paper/5b161280729dfb4707f4e0eb9ba65888e8391103/docs/montoro_structure.html" target="_blank">5b16128</a>
</td>
<td>
Jason Willwerscheid
</td>
<td>
2025-02-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/willwerscheid/ebmf-for-scrnaseq-paper/blob/61633b16feebc600834b1a4635cb305d749cae31/analysis/montoro_structure.Rmd" target="_blank">61633b1</a>
</td>
<td>
Jason Willwerscheid
</td>
<td>
2025-02-02
</td>
<td>
workflowr::wflow_publish("analysis/montoro_structure.Rmd")
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="introduction" class="section level2">
<h2>Introduction</h2>
<p>Here I produce structure plots for EBNMF, NMF, and topic model fits
to the larger dataset from Montoro et al.</p>
<p>Load the Tidyverse:</p>
<pre class="r"><code>library(tidyverse)
library(cowplot)
library(reactable)</code></pre>
<p>Load the <code>flashier</code> results:</p>
<pre class="r"><code>scale_FF &lt;- function(FF, LL, D = 1) {
  LL_norms &lt;- apply(LL, 2, function(x) sqrt(sum(x^2)))
  return(t(t(FF) * D * LL_norms))
}

ebnmf &lt;- readRDS(&quot;output/montoro-ebmf.rds&quot;)
ebnmf_FF &lt;- scale_FF(ebnmf$fit$F.pm, ebnmf$fit$L.pm)</code></pre>
<p>Extract cell covariates:</p>
<pre class="r"><code>cell_types &lt;- c(
      &quot;Ciliated&quot;,
      &quot;Proliferating&quot;,
      &quot;Basal&quot;,
      &quot;Club&quot;,
      &quot;Club (hillock-associated)&quot;,
      &quot;Goblet.progenitor&quot;,
      &quot;Goblet.1&quot;,
      &quot;Goblet.2&quot;,
      &quot;Tuft.progenitor&quot;,
      &quot;Tuft.1&quot;,
      &quot;Tuft.2&quot;,
      &quot;Neuroendocrine&quot;,
      &quot;Ionocyte&quot;
)

ct_abbr &lt;- cell_types
ct_abbr &lt;- str_replace(ct_abbr, &quot;\\.&quot;, &quot;\\-&quot;)
ct_abbr[2] &lt;- &quot;Prolif.&quot;
ct_abbr[5] &lt;- &quot;Hillock&quot;
ct_abbr[12] &lt;- &quot;NEC&quot;
ct_abbr &lt;- str_replace(ct_abbr, &quot;progenitor&quot;, &quot;Pr.&quot;)

external_info &lt;- tibble(
  CellType = sapply(strsplit(rownames(ebnmf_FF), &quot;_&quot;), `[[`, 5),
  TimePoint = sapply(strsplit(rownames(ebnmf_FF), &quot;_&quot;), `[[`, 2),
  Replicate = sapply(strsplit(rownames(ebnmf_FF), &quot;_&quot;), `[[`, 3)
) |&gt;
  left_join(tibble(CellType = cell_types, CtAbbr = ct_abbr), by = &quot;CellType&quot;) |&gt;
  mutate(
    CellType = factor(CellType, levels = cell_types),
    CtAbbr = factor(CtAbbr, levels = ct_abbr),
    TimePoint = factor(TimePoint),
    Replicate = factor(Replicate),
  ) |&gt;
  mutate(
    TPxRep = fct_cross(Replicate, TimePoint),
    CommonType = fct_collapse(CtAbbr, Other = ct_abbr[6:13])
  )</code></pre>
<p>I will use the following functions to generate plots:</p>
<pre class="r"><code>make_tib &lt;- function(FF, info_col, ksort, ncells = 50, duprate = 1) {
  info &lt;- external_info[[info_col]]

  # Downsample the number of cells.
  set.seed(666)
  cell_idx &lt;- numeric(0)
  for (k in levels(info)) {
    which_idx &lt;- which(info == k)
    # Downsample common cell types. Duplicate rare ones.
    if (ncells &lt; length(which_idx)) {
      which_idx &lt;- sample(which_idx, ncells, replace = FALSE)
    } else if (ncells &gt;= length(which_idx) * duprate) {
      which_idx &lt;- rep(which_idx, duprate)
    }
    cell_idx &lt;- c(cell_idx, which_idx)
  }

  FF &lt;- FF[cell_idx, ]
  info &lt;- info[cell_idx]
  colnames(FF) &lt;- 1:ncol(FF)

  lvls &lt;- as.numeric(ksort)
  if (any(setdiff(1:ncol(FF), ksort))) {
    lvls &lt;- c(&quot;Other&quot;, lvls)
  }

  tib &lt;- as_tibble(FF) |&gt;
    mutate(
      CellIdx = row_number(),
      Info = info
    ) |&gt;
    pivot_longer(
      -c(CellIdx, Info),
      names_to = &quot;Component&quot;,
      values_to = &quot;Loading&quot;,
      values_drop_na = TRUE
    ) |&gt;
    mutate(
      Component = ifelse(as.numeric(Component) %in% ksort, Component, &quot;Other&quot;)
    ) |&gt;
    group_by(CellIdx, Info, Component) |&gt;
    summarize(Loading = sum(Loading), .groups = &quot;drop&quot;) |&gt;
    mutate(
      Component = factor(Component, levels = lvls)
    ) |&gt;
    arrange(Info, Component)
  
  levels(tib$Component) &lt;- c(&quot;Other&quot;, names(ksort))

  return(tib)
}

do_plot &lt;- function(tib, colors, ptitle, ntiles = 400) {
  struct_df &lt;- tib |&gt;
    pivot_wider(names_from = Component, values_from = Loading)
  Lmat &lt;- struct_df |&gt;
    select(-(1:2)) |&gt;
    as.matrix()
  group &lt;- struct_df$Info
  
  if (&quot;Other&quot; %in% tib$Component) {
    colors &lt;- c(&quot;grey90&quot;, colors)
  }
  
  p &lt;- fastTopics::structure_plot(
    Lmat, topics = colnames(Lmat),
    grouping = group,
    colors = colors,
    # embed_method = fastTopics::umap_from_topics,
    # dims = 1,
    gap = 10,
    verbose = FALSE
  ) + theme(
    axis.text.y = element_blank()
  ) + labs(
    y = &quot;&quot;,
    title = ptitle
  )

  return(p)
}

glasbey &lt;- function(k) {
  return(fastTopics:::glasbey()[k])
}</code></pre>
<p>The following function is used to generate tables of gene set
annotations:</p>
<pre class="r"><code>make_tbl &lt;- function(GSEA, ksort) {
  klabs &lt;- tibble(
    korig = sort(ksort),
    ksort = ksort,
    label = factor(names(ksort), levels = names(ksort))
  )

  keep_res &lt;- GSEA |&gt;
    left_join(klabs, by = c(&quot;k&quot; = &quot;ksort&quot;)) |&gt;
    group_by(k) |&gt;
    filter(FDR &lt; 0.05 | rank(FDR) == min(rank(FDR))) |&gt;
    ungroup()

  shared_genesets &lt;- keep_res |&gt;
    left_join(keep_res |&gt; select(geneSet, shared_by = label),
              by = &quot;geneSet&quot;,
              relationship = &quot;many-to-many&quot;) |&gt;
    filter(shared_by != label) |&gt;
    arrange(label) |&gt;
    group_by(label, geneSet) |&gt;
    summarize(shared_by = paste(shared_by, collapse = &quot;,&quot;))
  
  keep_res &lt;- keep_res |&gt;
    select(label, geneSet, description, FDR) |&gt;
    left_join(shared_genesets, by = c(&quot;label&quot;, &quot;geneSet&quot;)) |&gt;
    arrange(label, FDR, description)
  
  keep_res &lt;- keep_res |&gt;
    mutate(FDR = ifelse(FDR &lt; 0.01, &quot;&lt;.01&quot;, format(round(FDR, 2), nsmall = 2)))
  
  react_tbl &lt;- reactable(
    keep_res, 
    columns = list(
      label = colDef(name = &quot;Component&quot;),
      geneSet = colDef(name = &quot;Gene Set&quot;),
      description = colDef(name = &quot;Description&quot;),
      shared_by = colDef(name = &quot;Shared By&quot;)
    ),
    groupBy = &quot;label&quot;
  )
  
  return(react_tbl)
}</code></pre>
</div>
<div id="ebnmf" class="section level2">
<h2>EBNMF</h2>
<p>In each case I manually sort and label factors.</p>
<pre class="r"><code>ebnmf_ksort &lt;- c(
  26, 27, 30, # Sparse
  1, 8, 10, 16, # Dense
  11, 12, 2, 24, # Time points
  6, 13, 22, # Ciliated
  21, 23, 17, # Proliferating
  4, 5, 9, # Basal
  3, 15, 7, # 26, Club-Hillock
  25, 28, 29, # Goblet
  18, 19, 14, 20 # Tuft-NEC-Ionocyte
)
names(ebnmf_ksort) &lt;- c(
  &quot;Sprs-a&quot;, &quot;Sprs-b&quot;, &quot;Sprs-c&quot;,
  &quot;Dns-a&quot;, &quot;Dns-b&quot;, &quot;Dns-c&quot;, &quot;Dns-d&quot;,
  &quot;Tp0&quot;, &quot;Tp30&quot;, &quot;Tp60&quot;, &quot;R1xTp30&quot;,
  &quot;Cil-a&quot;, &quot;Cil-b&quot;, &quot;Cil-c&quot;,
  &quot;Prl-a&quot;, &quot;Prl-b&quot;, &quot;Prl-c&quot;,
  &quot;Bas-a&quot;, &quot;Bas-b&quot;, &quot;Bas-c&quot;,
  &quot;Cl/G-a&quot;, &quot;Cl/G-b&quot;, &quot;Hil&quot;,
  &quot;Gob&quot;, &quot;Gob1&quot;, &quot;Gob2&quot;,
  &quot;Tuf1&quot;, &quot;Tuf2&quot;, &quot;Nec&quot;, &quot;Ion&quot;
)

sparse_title &lt;- &quot;Very sparse factors&quot;
dense_title &lt;- &quot;Dense factors&quot;
tp_title &lt;- &quot;Time point factors&quot;
common_title &lt;- &quot;Prevalent cell type factors&quot;
rare_title &lt;- &quot;Rare cell type factors (common cell types are downsampled)&quot;

sparse_k &lt;- 1:3
ebnmf_tib &lt;- make_tib(ebnmf_FF, &quot;CommonType&quot;, ebnmf_ksort[sparse_k], Inf, 1)
sparse_p &lt;- do_plot(ebnmf_tib, glasbey(2:4), sparse_title)

dense_k &lt;- 4:7
ebnmf_tib &lt;- make_tib(ebnmf_FF, &quot;CommonType&quot;, ebnmf_ksort[dense_k], Inf, 1)
dense_p &lt;- do_plot(ebnmf_tib, glasbey(5:8), dense_title)

tp_k &lt;- 8:11
ebnmf_tib &lt;- make_tib(ebnmf_FF, &quot;TPxRep&quot;, ebnmf_ksort[tp_k], Inf, 1)
tp_p &lt;- do_plot(ebnmf_tib, glasbey(14:17), tp_title)

common_k &lt;- c(12:14, 18:20, 23)
ebnmf_tib &lt;- make_tib(ebnmf_FF, &quot;CommonType&quot;, ebnmf_ksort[common_k], Inf, 1)
common_p &lt;- do_plot(ebnmf_tib, glasbey(21:27), common_title)

rare_k &lt;- c(15:17, 21:22, 24:30)
ebnmf_tib &lt;- make_tib(ebnmf_FF, &quot;CtAbbr&quot;, ebnmf_ksort[rare_k], 100, 3)
rare_p &lt;- do_plot(ebnmf_tib, glasbey(32:43), rare_title)

plot_grid(common_p, rare_p, tp_p, dense_p, sparse_p,  nrow = 5, ncol = 1)</code></pre>
<p><img src="figure/montoro_structure.Rmd/unnamed-chunk-6-1.png" width="768" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-6-1">
Past versions of unnamed-chunk-6-1.png
</button>
</p>
<div id="fig-unnamed-chunk-6-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/willwerscheid/ebmf-for-scrnaseq-paper/blob/2e18b60ce2723f5c4a9b6d0add2c8afc4e5793cd/docs/figure/montoro_structure.Rmd/unnamed-chunk-6-1.png" target="_blank">2e18b60</a>
</td>
<td>
Jason Willwerscheid
</td>
<td>
2025-02-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/willwerscheid/ebmf-for-scrnaseq-paper/blob/5b161280729dfb4707f4e0eb9ba65888e8391103/docs/figure/montoro_structure.Rmd/unnamed-chunk-6-1.png" target="_blank">5b16128</a>
</td>
<td>
Jason Willwerscheid
</td>
<td>
2025-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="nmf" class="section level2">
<h2>NMF</h2>
<pre class="r"><code>nnlm &lt;- readRDS(&quot;output/montoro-nmf.rds&quot;)
nnlm_FF &lt;- scale_FF(t(nnlm$fit$h), nnlm$fit$w, nnlm$fit$d)

nnlm_ksort &lt;- c(
  1, 28, 10, 26, # dense
  9, 5, # time points (dense)
  3, 11, 8, 7, # time points
  21, # replicate x TP
  15, 22, 20, # ciliated
  24, 12, 19, # prolif
  2, 6, 4, 14, 13, # basal
  25, 17, 23, # club/goblet
  18, 16, # hillock
  29, # tuft
  27, # NEC
  30 # ionocyte
)
names(nnlm_ksort) &lt;- c(
  &quot;Dns-a&quot;, &quot;Dns-b&quot;, &quot;Dns-c&quot;, &quot;Dns-d&quot;,
  &quot;TpxCl&quot;, &quot;Tp30/60&quot;,
  &quot;Tp0-a&quot;, &quot;Tp0-b&quot;, &quot;Tp30&quot;, &quot;Tp60&quot;,
  &quot;R1xTp30&quot;,
  &quot;Cil-a&quot;, &quot;Cil-b&quot;, &quot;Cil-c&quot;,  
  &quot;Prl-a&quot;, &quot;Prl-b&quot;, &quot;Prl-c&quot;,
  &quot;Bas-a&quot;, &quot;Bas-b&quot;, &quot;Bas-c&quot;, &quot;Bas-d&quot;, &quot;Bas-e&quot;,
  &quot;Cl/G-a&quot;, &quot;Cl/G-b&quot;, &quot;Cl/G-c&quot;,
  &quot;Hil-a&quot;, &quot;Hil-b&quot;,
  &quot;Tuf&quot;,
  &quot;Nec&quot;,
  &quot;Ion/Cl/G&quot;
)

nnlm_dense_k &lt;- 1:4
nnlm_tib &lt;- make_tib(nnlm_FF, &quot;CommonType&quot;, nnlm_ksort[nnlm_dense_k], Inf, 1)
nnlm_dense_p &lt;- do_plot(nnlm_tib, glasbey(5:8), dense_title)

nnlm_tp_k &lt;- 5:11
nnlm_tib &lt;- make_tib(nnlm_FF, &quot;TPxRep&quot;, nnlm_ksort[nnlm_tp_k], Inf, 1)
nnlm_tp_p &lt;- do_plot(nnlm_tib, glasbey(11:17), tp_title)

nnlm_common_k &lt;- c(12:14, 18:22, 26:27)
nnlm_tib &lt;- make_tib(nnlm_FF, &quot;CommonType&quot;, nnlm_ksort[nnlm_common_k], Inf, 1)
nnlm_common_p &lt;- do_plot(nnlm_tib, glasbey(21:30), common_title)

nnlm_rare_k &lt;- c(15:17, 23:25, 28:30)
nnlm_tib &lt;- make_tib(nnlm_FF, &quot;CtAbbr&quot;, nnlm_ksort[nnlm_rare_k], 100, 3)
nnlm_rare_p &lt;- do_plot(nnlm_tib, glasbey(32:40), rare_title)

plot_grid(nnlm_common_p, nnlm_rare_p, nnlm_tp_p, nnlm_dense_p, nrow = 4, ncol = 1)</code></pre>
<p><img src="figure/montoro_structure.Rmd/unnamed-chunk-7-1.png" width="768" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-7-1">
Past versions of unnamed-chunk-7-1.png
</button>
</p>
<div id="fig-unnamed-chunk-7-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/willwerscheid/ebmf-for-scrnaseq-paper/blob/2e18b60ce2723f5c4a9b6d0add2c8afc4e5793cd/docs/figure/montoro_structure.Rmd/unnamed-chunk-7-1.png" target="_blank">2e18b60</a>
</td>
<td>
Jason Willwerscheid
</td>
<td>
2025-02-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/willwerscheid/ebmf-for-scrnaseq-paper/blob/5b161280729dfb4707f4e0eb9ba65888e8391103/docs/figure/montoro_structure.Rmd/unnamed-chunk-7-1.png" target="_blank">5b16128</a>
</td>
<td>
Jason Willwerscheid
</td>
<td>
2025-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="topic-model" class="section level2">
<h2>Topic model</h2>
<pre class="r"><code>tm &lt;- readRDS(&quot;output/montoro-topics.rds&quot;)
tm_fit &lt;- tm$fit
tm_fit$F &lt;- tm$fit$L
tm_fit$L &lt;- tm$fit$F
tm_fit$Fn &lt;- tm$fit$Ln
tm_fit$Ln &lt;- tm$fit$Fn
tm_fit$Fy &lt;- tm$fit$Ly
tm_fit$Ly &lt;- tm$fit$Fy
tm_fit &lt;- fastTopics::poisson2multinom(tm_fit)
tm_FF &lt;- tm_fit$L

tm_ksort &lt;- c(
  21, 5, 11, 17, 27, # Dense  
  19, # Tp30/60xCl
  15, 8, # Tp0/60?
  24, # Tp30/60
  14, 26, # Tp0
  23, 29, 9, # Tp30 - 23 also club
  13, # Tp60
  25, # Ciliated
  6, # Proliferating
  10, 4, # Basal/Prolif
  12, # Basal
  3, # Club/Goblet
  18, # Club/Hillock
  22, # Club
  16, 28, # Hillock/Goblet-progenitor
  20, # Hillock
  7, # Goblet-p/1
  2, # Goblet-2
  1, 30 # Tuft-NEC-Ion
)
names(tm_ksort) &lt;- c(
  &quot;Dns-a&quot;, &quot;Dns-b&quot;, &quot;Dns-c&quot;,
  &quot;Dns-d&quot;, &quot;Dns-e&quot;,
  &quot;TpxCl&quot;,
  &quot;Tp0/60-a&quot;, &quot;Tp0/60-b&quot;,
  &quot;Tp30/60&quot;,
  &quot;Tp0-a&quot;, &quot;Tp0-b&quot;,
  &quot;Tp30xCl&quot;, &quot;Tp30-a&quot;, &quot;Tp30-b&quot;,
  &quot;Tp60&quot;,
  &quot;Cil&quot;,
  &quot;Prl&quot;,
  &quot;B/P-a&quot;, &quot;B/P-b&quot;,
  &quot;Bas&quot;, 
  &quot;Cl/G&quot;, &quot;Cl/Hil&quot;, &quot;Club&quot;,
  &quot;H/G-a&quot;, &quot;H/G-b&quot;, 
  &quot;Hil&quot;, 
  &quot;Gob1&quot;, &quot;Gob2&quot;,
  &quot;TNI-a&quot;, &quot;TNI-b&quot;
)

tm_dense_k &lt;- 1:5
tm_tib &lt;- make_tib(tm_FF, &quot;CommonType&quot;, tm_ksort[tm_dense_k], Inf, 1)
tm_dense_p &lt;- do_plot(tm_tib, glasbey(5:9), dense_title)

tm_tp_k &lt;- 6:15
tm_tib &lt;- make_tib(tm_FF, &quot;TPxRep&quot;, tm_ksort[tm_tp_k], Inf, 1)
tm_tp_p &lt;- do_plot(tm_tib, glasbey(11:20), tp_title)

tm_common_k &lt;- c(16, 20, 22:23, 26)
tm_tib &lt;- make_tib(tm_FF, &quot;CommonType&quot;, tm_ksort[tm_common_k], Inf, 1)
tm_common_p &lt;- do_plot(tm_tib, glasbey(24:29), common_title)

tm_rare_k &lt;- c(17:19, 21, 24:25, 27:30)
tm_tib &lt;- make_tib(tm_FF, &quot;CtAbbr&quot;, tm_ksort[tm_rare_k], 100, 3)
tm_rare_p &lt;- do_plot(tm_tib, glasbey(32:41), rare_title)

plot_grid(tm_common_p, tm_rare_p, tm_tp_p, tm_dense_p, nrow = 4, ncol = 1)</code></pre>
<p><img src="figure/montoro_structure.Rmd/unnamed-chunk-8-1.png" width="768" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-8-1">
Past versions of unnamed-chunk-8-1.png
</button>
</p>
<div id="fig-unnamed-chunk-8-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/willwerscheid/ebmf-for-scrnaseq-paper/blob/2e18b60ce2723f5c4a9b6d0add2c8afc4e5793cd/docs/figure/montoro_structure.Rmd/unnamed-chunk-8-1.png" target="_blank">2e18b60</a>
</td>
<td>
Jason Willwerscheid
</td>
<td>
2025-02-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="ebnmf-annotations" class="section level2">
<h2>EBNMF annotations</h2>
<p>I use GSEA to do annotations. For each component, I provide as input
all genes with loadings that are larger than 0.01 times the maximum
loading (for the topic model, 0.001 times the maximum loading). (If all
loadings are provided, GSEA will also find gene sets with very small
loadings across genes, which is obviously not what we want.) I report
the top 5 gene sets with FDR &lt; 0.05 for each component (ties are
included), with the restriction that at least one gene set is reported
for each factor (even if its FDR is greater than 0.05).</p>
<pre class="r"><code>ebnmf_GSEA &lt;- readRDS(&quot;output/montoro-ebnmf-gsea.rds&quot;)
make_tbl(ebnmf_GSEA, ebnmf_ksort[c(12:30, 8:11, 4:7, 1:3)])</code></pre>
<div class="reactable html-widget html-fill-item" id="htmlwidget-249d5df492545401a4d6" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-249d5df492545401a4d6">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"label":["Cil-a","Cil-a","Cil-a","Cil-a","Cil-b","Cil-c","Cil-c","Cil-c","Cil-c","Cil-c","Cil-c","Cil-c","Cil-c","Cil-c","Cil-c","Cil-c","Cil-c","Cil-c","Cil-c","Cil-c","Cil-c","Cil-c","Cil-c","Cil-c","Cil-c","Prl-a","Prl-a","Prl-a","Prl-a","Prl-a","Prl-a","Prl-a","Prl-a","Prl-a","Prl-a","Prl-a","Prl-a","Prl-a","Prl-a","Prl-a","Prl-a","Prl-a","Prl-a","Prl-a","Prl-b","Prl-b","Prl-b","Prl-b","Prl-b","Prl-b","Prl-c","Prl-c","Bas-a","Bas-a","Bas-a","Bas-a","Bas-a","Bas-a","Bas-a","Bas-a","Bas-b","Bas-b","Bas-c","Cl/G-a","Cl/G-a","Cl/G-b","Cl/G-b","Cl/G-b","Hil","Hil","Hil","Hil","Hil","Gob","Gob1","Gob2","Tuf1","Tuf1","Tuf1","Tuf1","Tuf1","Tuf1","Tuf1","Tuf1","Tuf1","Tuf1","Tuf1","Tuf1","Tuf1","Tuf1","Tuf1","Tuf2","Tuf2","Tuf2","Tuf2","Tuf2","Tuf2","Tuf2","Tuf2","Tuf2","Tuf2","Nec","Nec","Nec","Nec","Nec","Nec","Nec","Nec","Nec","Nec","Nec","Ion","Ion","Ion","Ion","Ion","Ion","Ion","Ion","Ion","Ion","Ion","Tp0","Tp0","Tp0","Tp0","Tp0","Tp0","Tp0","Tp0","Tp0","Tp0","Tp30","Tp30","Tp30","Tp30","Tp30","Tp30","Tp30","Tp30","Tp30","Tp30","Tp60","Tp60","R1xTp30","R1xTp30","R1xTp30","R1xTp30","R1xTp30","R1xTp30","R1xTp30","R1xTp30","R1xTp30","R1xTp30","R1xTp30","R1xTp30","R1xTp30","Dns-a","Dns-a","Dns-a","Dns-a","Dns-a","Dns-a","Dns-a","Dns-a","Dns-a","Dns-a","Dns-a","Dns-a","Dns-a","Dns-a","Dns-a","Dns-b","Dns-b","Dns-b","Dns-b","Dns-b","Dns-b","Dns-b","Dns-b","Dns-b","Dns-b","Dns-b","Dns-c","Dns-c","Dns-c","Dns-c","Dns-c","Dns-c","Dns-c","Dns-c","Dns-c","Dns-d","Sprs-a","Sprs-a","Sprs-a","Sprs-a","Sprs-a","Sprs-a","Sprs-a","Sprs-a","Sprs-a","Sprs-b","Sprs-c"],"geneSet":["GO:0030534","GO:0060048","GO:0055117","GO:0009408","GO:0007569","GO:0032922","GO:0032964","GO:0032963","GO:0016614","GO:0016616","GO:0006090","GO:0032965","GO:0010712","GO:0060420","GO:0046620","GO:1904589","GO:0042306","GO:0044445","GO:0022627","GO:0098800","GO:0015935","GO:0044391","GO:0005840","GO:0022626","GO:0003735","GO:0035869","GO:0030879","GO:0050954","GO:0004930","GO:0007368","GO:0032391","GO:0001974","GO:0005840","GO:0008589","GO:0044391","GO:0015934","GO:0016125","GO:0022626","GO:0005198","GO:0031461","GO:0003735","GO:0022625","GO:1902911","GO:1902554","GO:0009855","GO:0007368","GO:0009799","GO:0042393","GO:0005930","GO:0097014","GO:0005882","GO:0045111","GO:0097731","GO:0097733","GO:0035869","GO:0016573","GO:0030990","GO:0048636","GO:0045844","GO:0097730","GO:0008237","GO:0097730","GO:0032391","GO:0046578","GO:0051056","GO:1903578","GO:0006140","GO:1900542","GO:0042752","GO:0019730","GO:0006959","GO:0045111","GO:0005882","GO:0090407","GO:0033674","GO:0006996","GO:0044445","GO:0022626","GO:0022627","GO:0098800","GO:0098798","GO:0044391","GO:0005840","GO:0015935","GO:0003735","GO:0070469","GO:0000793","GO:0042089","GO:0042107","GO:0006835","GO:0007224","GO:0022625","GO:0044445","GO:0022626","GO:0022627","GO:0015934","GO:0044391","GO:0005840","GO:0015935","GO:0003735","GO:0005198","GO:0005088","GO:0022625","GO:0044445","GO:0022626","GO:0022627","GO:0044391","GO:0005840","GO:0003735","GO:0015935","GO:0015934","GO:0098800","GO:0015078","GO:0009055","GO:1902600","GO:0036442","GO:0016469","GO:0015077","GO:0044769","GO:0035023","GO:0002822","GO:0022853","GO:0019829","GO:0022627","GO:0060021","GO:0003735","GO:0015935","GO:0044391","GO:0022626","GO:0005840","GO:0042274","GO:0044445","GO:0005198","GO:0097731","GO:0035869","GO:0008234","GO:0051606","GO:0097730","GO:0097733","GO:0016860","GO:0009135","GO:0009179","GO:0009185","GO:0043620","GO:0043618","GO:0044445","GO:0022626","GO:0022627","GO:0002821","GO:0044391","GO:0005840","GO:0015935","GO:0003735","GO:0005791","GO:0043043","GO:0032386","GO:1905039","GO:1903825","GO:0043604","GO:0043603","GO:0044445","GO:0022626","GO:0001822","GO:0007162","GO:1901566","GO:0043043","GO:0006518","GO:0072001","GO:1990904","GO:0005840","GO:0006412","GO:0001655","GO:0051961","GO:0031072","GO:0034470","GO:0007018","GO:0030705","GO:0034605","GO:0006606","GO:0009266","GO:0009267","GO:0009408","GO:0051170","GO:0006986","GO:0004364","GO:0030990","GO:0072341","GO:0060828","GO:0016765","GO:0071466","GO:0006805","GO:0030111","GO:0048306","GO:0008509","GO:0044445","GO:0022626","GO:0034660","GO:0044391","GO:0005840","GO:0003735","GO:0016072","GO:0022625","GO:0015934","GO:0003008","GO:0043087"],"description":["adult behavior","cardiac muscle contraction","regulation of cardiac muscle contraction","response to heat","cell aging","circadian regulation of gene expression","collagen biosynthetic process","collagen metabolic process","oxidoreductase activity, acting on CH-OH group of donors","oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor","pyruvate metabolic process","regulation of collagen biosynthetic process","regulation of collagen metabolic process","regulation of heart growth","regulation of organ growth","regulation of protein import","regulation of protein import into nucleus","cytosolic part","cytosolic small ribosomal subunit","inner mitochondrial membrane protein complex","small ribosomal subunit","ribosomal subunit","ribosome","cytosolic ribosome","structural constituent of ribosome","ciliary transition zone","mammary gland development","sensory perception of mechanical stimulus","G protein-coupled receptor activity","determination of left/right symmetry","photoreceptor connecting cilium","blood vessel remodeling","ribosome","regulation of smoothened signaling pathway","ribosomal subunit","large ribosomal subunit","sterol metabolic process","cytosolic ribosome","structural molecule activity","cullin-RING ubiquitin ligase complex","structural constituent of ribosome","cytosolic large ribosomal subunit","protein kinase complex","serine/threonine protein kinase complex","determination of bilateral symmetry","determination of left/right symmetry","specification of symmetry","histone binding","axoneme","ciliary plasm","intermediate filament","intermediate filament cytoskeleton","9+0 non-motile cilium","photoreceptor cell cilium","ciliary transition zone","histone acetylation","intraciliary transport particle","positive regulation of muscle organ development","positive regulation of striated muscle tissue development","non-motile cilium","metallopeptidase activity","non-motile cilium","photoreceptor connecting cilium","regulation of Ras protein signal transduction","regulation of small GTPase mediated signal transduction","regulation of ATP metabolic process","regulation of nucleotide metabolic process","regulation of purine nucleotide metabolic process","regulation of circadian rhythm","antimicrobial humoral response","humoral immune response","intermediate filament cytoskeleton","intermediate filament","organophosphate biosynthetic process","positive regulation of kinase activity","organelle organization","cytosolic part","cytosolic ribosome","cytosolic small ribosomal subunit","inner mitochondrial membrane protein complex","mitochondrial protein complex","ribosomal subunit","ribosome","small ribosomal subunit","structural constituent of ribosome","respiratory chain","condensed chromosome","cytokine biosynthetic process","cytokine metabolic process","dicarboxylic acid transport","smoothened signaling pathway","cytosolic large ribosomal subunit","cytosolic part","cytosolic ribosome","cytosolic small ribosomal subunit","large ribosomal subunit","ribosomal subunit","ribosome","small ribosomal subunit","structural constituent of ribosome","structural molecule activity","Ras guanyl-nucleotide exchange factor activity","cytosolic large ribosomal subunit","cytosolic part","cytosolic ribosome","cytosolic small ribosomal subunit","ribosomal subunit","ribosome","structural constituent of ribosome","small ribosomal subunit","large ribosomal subunit","inner mitochondrial membrane protein complex","proton transmembrane transporter activity","electron transfer activity","proton transmembrane transport","proton-exporting ATPase activity","proton-transporting two-sector ATPase complex","monovalent inorganic cation transmembrane transporter activity","ATPase activity, coupled to transmembrane movement of ions, rotational mechanism","regulation of Rho protein signal transduction","regulation of adaptive immune response based on somatic recombination of immune receptors built from immunoglobulin superfamily domains","active ion transmembrane transporter activity","cation-transporting ATPase activity","cytosolic small ribosomal subunit","roof of mouth development","structural constituent of ribosome","small ribosomal subunit","ribosomal subunit","cytosolic ribosome","ribosome","ribosomal small subunit biogenesis","cytosolic part","structural molecule activity","9+0 non-motile cilium","ciliary transition zone","cysteine-type peptidase activity","detection of stimulus","non-motile cilium","photoreceptor cell cilium","intramolecular oxidoreductase activity","purine nucleoside diphosphate metabolic process","purine ribonucleoside diphosphate metabolic process","ribonucleoside diphosphate metabolic process","regulation of DNA-templated transcription in response to stress","regulation of transcription from RNA polymerase II promoter in response to stress","cytosolic part","cytosolic ribosome","cytosolic small ribosomal subunit","positive regulation of adaptive immune response","ribosomal subunit","ribosome","small ribosomal subunit","structural constituent of ribosome","rough endoplasmic reticulum","peptide biosynthetic process","regulation of intracellular transport","carboxylic acid transmembrane transport","organic acid transmembrane transport","amide biosynthetic process","cellular amide metabolic process","cytosolic part","cytosolic ribosome","kidney development","negative regulation of cell adhesion","organonitrogen compound biosynthetic process","peptide biosynthetic process","peptide metabolic process","renal system development","ribonucleoprotein complex","ribosome","translation","urogenital system development","negative regulation of nervous system development","heat shock protein binding","ncRNA processing","microtubule-based movement","cytoskeleton-dependent intracellular transport","cellular response to heat","protein import into nucleus","response to temperature stimulus","cellular response to starvation","response to heat","import into nucleus","response to unfolded protein","glutathione transferase activity","intraciliary transport particle","modified amino acid binding","regulation of canonical Wnt signaling pathway","transferase activity, transferring alkyl or aryl (other than methyl) groups","cellular response to xenobiotic stimulus","xenobiotic metabolic process","regulation of Wnt signaling pathway","calcium-dependent protein binding","anion transmembrane transporter activity","cytosolic part","cytosolic ribosome","ncRNA metabolic process","ribosomal subunit","ribosome","structural constituent of ribosome","rRNA metabolic process","cytosolic large ribosomal subunit","large ribosomal subunit","system process","regulation of GTPase activity"],"FDR":["<.01","0.04","0.05","0.05","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","0.01","0.01","0.01","0.02","0.02","<.01","<.01","<.01","<.01","0.01","0.01","0.02","0.02","0.02","0.02","0.02","0.03","0.03","0.03","0.03","0.03","0.04","0.04","0.04","<.01","<.01","<.01","<.01","0.02","0.02","<.01","<.01","0.02","0.02","0.02","0.02","0.02","0.03","0.03","0.04","<.01","0.03","<.01","<.01","<.01","0.05","0.05","0.05","<.01","0.02","0.02","0.02","0.04","<.01","0.05","0.14","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","0.02","0.02","0.02","0.02","0.03","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","0.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","0.01","0.01","0.02","0.02","<.01","<.01","0.01","0.01","0.01","0.01","0.02","0.02","0.04","0.05","<.01","<.01","<.01","<.01","<.01","<.01","<.01","0.01","0.01","0.01","0.03","0.03","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","0.01","0.01","0.03","0.04","0.04","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","0.01","<.01","<.01","<.01","<.01","0.02","0.02","0.02","0.02","0.03","0.03","0.03","<.01","<.01","<.01","<.01","<.01","0.01","0.02","0.03","0.04","0.21","<.01","<.01","<.01","<.01","<.01","<.01","0.02","0.04","0.04","<.01","<.01"],"shared_by":[null,null,null,"Dns-b",null,null,null,null,null,null,null,null,null,null,null,null,null,"Dns-a,Tp0,Nec,Tuf1,Tuf2,R1xTp30,Sprs-a","Tp0,Nec,Tuf1,Tuf2,R1xTp30","Nec,Tuf1","Tp0,Nec,Tuf1,Tuf2,R1xTp30","Tp0,Nec,Tuf1,Tuf2,Prl-a,R1xTp30,Sprs-a","Dns-a,Tp0,Nec,Tuf1,Tuf2,Prl-a,R1xTp30,Sprs-a","Dns-a,Tp0,Nec,Tuf1,Tuf2,Prl-a,R1xTp30,Sprs-a","Tp0,Nec,Tuf1,Tuf2,Prl-a,R1xTp30,Sprs-a","Bas-a,Tp30",null,null,null,"Prl-b","Bas-c",null,"Dns-a,Tp0,Nec,Tuf1,Tuf2,Cil-c,R1xTp30,Sprs-a",null,"Tp0,Nec,Tuf1,Tuf2,Cil-c,R1xTp30,Sprs-a","Nec,Tuf2,Sprs-a",null,"Dns-a,Tp0,Nec,Tuf1,Tuf2,Cil-c,R1xTp30,Sprs-a","Tp0,Tuf2",null,"Tp0,Nec,Tuf1,Tuf2,Cil-c,R1xTp30,Sprs-a","Nec,Tuf2,Sprs-a",null,null,null,"Prl-a",null,null,null,null,"Hil","Hil","Tp30","Tp30","Tp30,Prl-a",null,"Dns-c",null,null,"Bas-b,Tp30",null,"Bas-a,Tp30","Prl-a",null,null,null,null,null,null,null,null,"Prl-c","Prl-c",null,null,null,"Dns-a,Tp0,Nec,Tuf2,Cil-c,R1xTp30,Sprs-a","Dns-a,Tp0,Nec,Tuf2,Prl-a,Cil-c,R1xTp30,Sprs-a","Tp0,Nec,Tuf2,Cil-c,R1xTp30","Nec,Cil-c",null,"Tp0,Nec,Tuf2,Prl-a,Cil-c,R1xTp30,Sprs-a","Dns-a,Tp0,Nec,Tuf2,Prl-a,Cil-c,R1xTp30,Sprs-a","Tp0,Nec,Tuf2,Cil-c,R1xTp30","Tp0,Nec,Tuf2,Prl-a,Cil-c,R1xTp30,Sprs-a",null,null,null,null,null,null,"Nec,Prl-a,Sprs-a","Dns-a,Tp0,Nec,Tuf1,Cil-c,R1xTp30,Sprs-a","Dns-a,Tp0,Nec,Tuf1,Prl-a,Cil-c,R1xTp30,Sprs-a","Tp0,Nec,Tuf1,Cil-c,R1xTp30","Nec,Prl-a,Sprs-a","Tp0,Nec,Tuf1,Prl-a,Cil-c,R1xTp30,Sprs-a","Dns-a,Tp0,Nec,Tuf1,Prl-a,Cil-c,R1xTp30,Sprs-a","Tp0,Nec,Tuf1,Cil-c,R1xTp30","Tp0,Nec,Tuf1,Prl-a,Cil-c,R1xTp30,Sprs-a","Tp0,Prl-a",null,"Tuf2,Prl-a,Sprs-a","Dns-a,Tp0,Tuf1,Tuf2,Cil-c,R1xTp30,Sprs-a","Dns-a,Tp0,Tuf1,Tuf2,Prl-a,Cil-c,R1xTp30,Sprs-a","Tp0,Tuf1,Tuf2,Cil-c,R1xTp30","Tp0,Tuf1,Tuf2,Prl-a,Cil-c,R1xTp30,Sprs-a","Dns-a,Tp0,Tuf1,Tuf2,Prl-a,Cil-c,R1xTp30,Sprs-a","Tp0,Tuf1,Tuf2,Prl-a,Cil-c,R1xTp30,Sprs-a","Tp0,Tuf1,Tuf2,Cil-c,R1xTp30","Tuf2,Prl-a,Sprs-a","Tuf1,Cil-c",null,null,null,null,null,null,null,null,null,null,null,"Nec,Tuf1,Tuf2,Cil-c,R1xTp30",null,"Nec,Tuf1,Tuf2,Prl-a,Cil-c,R1xTp30,Sprs-a","Nec,Tuf1,Tuf2,Cil-c,R1xTp30","Nec,Tuf1,Tuf2,Prl-a,Cil-c,R1xTp30,Sprs-a","Dns-a,Nec,Tuf1,Tuf2,Prl-a,Cil-c,R1xTp30,Sprs-a","Dns-a,Nec,Tuf1,Tuf2,Prl-a,Cil-c,R1xTp30,Sprs-a",null,"Dns-a,Nec,Tuf1,Tuf2,Cil-c,R1xTp30,Sprs-a","Tuf2,Prl-a","Bas-a","Bas-a,Prl-a",null,null,"Bas-a,Bas-b","Bas-a",null,null,null,null,null,null,"Dns-a,Tp0,Nec,Tuf1,Tuf2,Cil-c,Sprs-a","Dns-a,Tp0,Nec,Tuf1,Tuf2,Prl-a,Cil-c,Sprs-a","Tp0,Nec,Tuf1,Tuf2,Cil-c",null,"Tp0,Nec,Tuf1,Tuf2,Prl-a,Cil-c,Sprs-a","Dns-a,Tp0,Nec,Tuf1,Tuf2,Prl-a,Cil-c,Sprs-a","Tp0,Nec,Tuf1,Tuf2,Cil-c","Tp0,Nec,Tuf1,Tuf2,Prl-a,Cil-c,Sprs-a",null,"Dns-a",null,null,null,null,null,"Tp0,Nec,Tuf1,Tuf2,Cil-c,R1xTp30,Sprs-a","Tp0,Nec,Tuf1,Tuf2,Prl-a,Cil-c,R1xTp30,Sprs-a",null,null,null,"R1xTp30",null,null,null,"Tp0,Nec,Tuf1,Tuf2,Prl-a,Cil-c,R1xTp30,Sprs-a",null,null,null,null,null,null,null,null,null,null,null,"Cil-a",null,null,null,"Bas-a",null,null,null,null,null,null,null,null,"Dns-a,Tp0,Nec,Tuf1,Tuf2,Cil-c,R1xTp30","Dns-a,Tp0,Nec,Tuf1,Tuf2,Prl-a,Cil-c,R1xTp30",null,"Tp0,Nec,Tuf1,Tuf2,Prl-a,Cil-c,R1xTp30","Dns-a,Tp0,Nec,Tuf1,Tuf2,Prl-a,Cil-c,R1xTp30","Tp0,Nec,Tuf1,Tuf2,Prl-a,Cil-c,R1xTp30",null,"Nec,Tuf2,Prl-a","Nec,Tuf2,Prl-a",null,null]},"columns":[{"id":"label","name":"Component","type":"factor"},{"id":"geneSet","name":"Gene Set","type":"character"},{"id":"description","name":"Description","type":"character"},{"id":"FDR","name":"FDR","type":"character"},{"id":"shared_by","name":"Shared By","type":"character"}],"groupBy":["label"],"dataKey":"cfc06b5982e185c3ca9346b720a4b980"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="nmf-annotations" class="section level2">
<h2>NMF annotations</h2>
<pre class="r"><code>nmf_GSEA &lt;- readRDS(&quot;output/montoro-nmf-gsea.rds&quot;)
make_tbl(nmf_GSEA, nnlm_ksort[c(12:30, 5:11, 1:4)])</code></pre>
<div class="reactable html-widget html-fill-item" id="htmlwidget-c3bb3c3afaa50f958c2d" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-c3bb3c3afaa50f958c2d">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"label":["Cil-a","Cil-b","Cil-c","Cil-c","Cil-c","Cil-c","Cil-c","Cil-c","Cil-c","Prl-a","Prl-a","Prl-a","Prl-a","Prl-b","Prl-b","Prl-b","Prl-b","Prl-b","Prl-b","Prl-b","Prl-b","Prl-b","Prl-c","Bas-a","Bas-a","Bas-a","Bas-a","Bas-a","Bas-a","Bas-a","Bas-a","Bas-a","Bas-a","Bas-b","Bas-b","Bas-b","Bas-b","Bas-b","Bas-b","Bas-b","Bas-b","Bas-b","Bas-b","Bas-b","Bas-b","Bas-b","Bas-b","Bas-c","Bas-c","Bas-c","Bas-c","Bas-c","Bas-d","Bas-e","Bas-e","Cl/G-a","Cl/G-a","Cl/G-a","Cl/G-a","Cl/G-a","Cl/G-a","Cl/G-a","Cl/G-a","Cl/G-a","Cl/G-a","Cl/G-b","Cl/G-b","Cl/G-b","Cl/G-b","Cl/G-b","Cl/G-c","Cl/G-c","Cl/G-c","Cl/G-c","Cl/G-c","Hil-a","Hil-a","Hil-b","Hil-b","Hil-b","Hil-b","Hil-b","Hil-b","Hil-b","Hil-b","Hil-b","Tuf","Tuf","Tuf","Tuf","Tuf","Tuf","Tuf","Tuf","Tuf","Tuf","Tuf","Tuf","Tuf","Tuf","Tuf","Tuf","Tuf","Nec","Nec","Ion/Cl/G","Ion/Cl/G","Ion/Cl/G","Ion/Cl/G","Ion/Cl/G","Ion/Cl/G","Ion/Cl/G","Ion/Cl/G","Ion/Cl/G","Ion/Cl/G","TpxCl","TpxCl","Tp30/60","Tp0-a","Tp0-a","Tp0-a","Tp0-a","Tp0-a","Tp0-a","Tp0-a","Tp0-a","Tp0-a","Tp0-a","Tp0-a","Tp0-a","Tp0-b","Tp0-b","Tp30","Tp60","Tp60","Tp60","Tp60","R1xTp30","R1xTp30","Dns-a","Dns-a","Dns-a","Dns-a","Dns-a","Dns-a","Dns-a","Dns-a","Dns-a","Dns-a","Dns-b","Dns-c","Dns-c","Dns-c","Dns-c","Dns-c","Dns-c","Dns-d","Dns-d","Dns-d","Dns-d","Dns-d","Dns-d","Dns-d","Dns-d","Dns-d","Dns-d","Dns-d","Dns-d","Dns-d"],"geneSet":["GO:0003300","GO:0042274","GO:0098916","GO:0007268","GO:0099536","GO:0099537","GO:0021537","GO:0050804","GO:0099177","GO:0030992","GO:0004888","GO:0007224","GO:0048771","GO:0015935","GO:0044445","GO:0022627","GO:0005840","GO:0044391","GO:0022626","GO:0005198","GO:0003735","GO:0030178","GO:0006364","GO:0022625","GO:0044445","GO:0022626","GO:0022627","GO:0015934","GO:0019843","GO:0044391","GO:0005840","GO:0015935","GO:0003735","GO:0030141","GO:0035914","GO:0060538","GO:0007519","GO:0005840","GO:1990904","GO:0005198","GO:0044391","GO:0044445","GO:0051592","GO:0009612","GO:0051650","GO:0051648","GO:0044282","GO:0015267","GO:0022803","GO:0022838","GO:0035869","GO:0042073","GO:0003714","GO:0001508","GO:0098800","GO:0061077","GO:0006457","GO:0042176","GO:0051082","GO:1903362","GO:0030163","GO:0009057","GO:1901565","GO:0033267","GO:0035966","GO:0022627","GO:0015935","GO:0021537","GO:0003714","GO:0045727","GO:0001558","GO:0010594","GO:0090407","GO:0030307","GO:0016049","GO:0005791","GO:0001818","GO:0006164","GO:0009152","GO:0072522","GO:0098800","GO:0098798","GO:0044455","GO:0005743","GO:0006403","GO:0019866","GO:0042089","GO:0042107","GO:0003735","GO:0044445","GO:0022626","GO:0098800","GO:0035249","GO:0098798","GO:0005840","GO:0044391","GO:0009055","GO:0044455","GO:0070469","GO:0009953","GO:0045178","GO:0035869","GO:0000793","GO:0005088","GO:1901605","GO:0070201","GO:0090087","GO:0051223","GO:0006511","GO:0051271","GO:0042886","GO:0015833","GO:0015031","GO:0016323","GO:0032386","GO:0043620","GO:0043618","GO:0001894","GO:0044445","GO:0022626","GO:0022627","GO:0043583","GO:0044391","GO:0005840","GO:0003735","GO:0015935","GO:0005198","GO:0022625","GO:0019843","GO:0072089","GO:0035869","GO:0030990","GO:0046777","GO:0050807","GO:0050803","GO:0042737","GO:0099173","GO:0002263","GO:0002366","GO:0043604","GO:0043603","GO:0044445","GO:0022626","GO:0043043","GO:0006518","GO:1990904","GO:0044391","GO:0005840","GO:0006412","GO:0009986","GO:0048037","GO:0034660","GO:0030111","GO:0006364","GO:0034470","GO:0016072","GO:0044445","GO:0022626","GO:0060047","GO:0003015","GO:0010817","GO:0044391","GO:0005840","GO:0003735","GO:0005198","GO:1990904","GO:0022627","GO:0043604","GO:0006412"],"description":["cardiac muscle hypertrophy","ribosomal small subunit biogenesis","anterograde trans-synaptic signaling","chemical synaptic transmission","synaptic signaling","trans-synaptic signaling","telencephalon development","modulation of chemical synaptic transmission","regulation of trans-synaptic signaling","intraciliary transport particle B","transmembrane signaling receptor activity","smoothened signaling pathway","tissue remodeling","small ribosomal subunit","cytosolic part","cytosolic small ribosomal subunit","ribosome","ribosomal subunit","cytosolic ribosome","structural molecule activity","structural constituent of ribosome","negative regulation of Wnt signaling pathway","rRNA processing","cytosolic large ribosomal subunit","cytosolic part","cytosolic ribosome","cytosolic small ribosomal subunit","large ribosomal subunit","rRNA binding","ribosomal subunit","ribosome","small ribosomal subunit","structural constituent of ribosome","secretory granule","skeletal muscle cell differentiation","skeletal muscle organ development","skeletal muscle tissue development","ribosome","ribonucleoprotein complex","structural molecule activity","ribosomal subunit","cytosolic part","response to calcium ion","response to mechanical stimulus","establishment of vesicle localization","vesicle localization","small molecule catabolic process","channel activity","passive transmembrane transporter activity","substrate-specific channel activity","ciliary transition zone","intraciliary transport","transcription corepressor activity","action potential","inner mitochondrial membrane protein complex","chaperone-mediated protein folding","protein folding","regulation of protein catabolic process","unfolded protein binding","regulation of cellular protein catabolic process","protein catabolic process","macromolecule catabolic process","organonitrogen compound catabolic process","axon part","response to topologically incorrect protein","cytosolic small ribosomal subunit","small ribosomal subunit","telencephalon development","transcription corepressor activity","positive regulation of translation","regulation of cell growth","regulation of endothelial cell migration","organophosphate biosynthetic process","positive regulation of cell growth","cell growth","rough endoplasmic reticulum","negative regulation of cytokine production","purine nucleotide biosynthetic process","purine ribonucleotide biosynthetic process","purine-containing compound biosynthetic process","inner mitochondrial membrane protein complex","mitochondrial protein complex","mitochondrial membrane part","mitochondrial inner membrane","RNA localization","organelle inner membrane","cytokine biosynthetic process","cytokine metabolic process","structural constituent of ribosome","cytosolic part","cytosolic ribosome","inner mitochondrial membrane protein complex","synaptic transmission, glutamatergic","mitochondrial protein complex","ribosome","ribosomal subunit","electron transfer activity","mitochondrial membrane part","respiratory chain","dorsal/ventral pattern formation","basal part of cell","ciliary transition zone","condensed chromosome","Ras guanyl-nucleotide exchange factor activity","alpha-amino acid metabolic process","regulation of establishment of protein localization","regulation of peptide transport","regulation of protein transport","ubiquitin-dependent protein catabolic process","negative regulation of cellular component movement","amide transport","peptide transport","protein transport","basolateral plasma membrane","regulation of intracellular transport","regulation of DNA-templated transcription in response to stress","regulation of transcription from RNA polymerase II promoter in response to stress","tissue homeostasis","cytosolic part","cytosolic ribosome","cytosolic small ribosomal subunit","ear development","ribosomal subunit","ribosome","structural constituent of ribosome","small ribosomal subunit","structural molecule activity","cytosolic large ribosomal subunit","rRNA binding","stem cell proliferation","ciliary transition zone","intraciliary transport particle","protein autophosphorylation","regulation of synapse organization","regulation of synapse structure or activity","drug catabolic process","postsynapse organization","cell activation involved in immune response","leukocyte activation involved in immune response","amide biosynthetic process","cellular amide metabolic process","cytosolic part","cytosolic ribosome","peptide biosynthetic process","peptide metabolic process","ribonucleoprotein complex","ribosomal subunit","ribosome","translation","cell surface","cofactor binding","ncRNA metabolic process","regulation of Wnt signaling pathway","rRNA processing","ncRNA processing","rRNA metabolic process","cytosolic part","cytosolic ribosome","heart contraction","heart process","regulation of hormone levels","ribosomal subunit","ribosome","structural constituent of ribosome","structural molecule activity","ribonucleoprotein complex","cytosolic small ribosomal subunit","amide biosynthetic process","translation"],"FDR":["<.01","0.06","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","0.02","0.05","<.01","<.01","<.01","<.01","<.01","0.01","0.02","0.03","0.03","0.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","0.01","0.01","0.03","<.01","<.01","<.01","0.01","0.02","0.08","<.01","0.05","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","0.01","<.01","<.01","<.01","0.02","0.03","<.01","0.04","0.03","0.03","0.03","0.04","0.04","0.04","0.04","0.05","0.05","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","0.02","0.02","0.03","0.03","0.03","0.04","<.01","<.01","<.01","<.01","<.01","<.01","<.01","0.02","0.02","0.02","0.03","0.04","0.13","0.13","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","0.02","0.01","0.02","<.01","0.02","0.02","0.03","0.04","0.59","0.59","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","0.06","<.01","<.01","<.01","0.03","0.04","0.05","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01"],"shared_by":[null,null,null,null,null,null,"Cl/G-b",null,null,null,null,null,null,"Bas-a,Tp0-a,Cl/G-b","Dns-a,Bas-a,Tp0-a,Bas-b,Dns-d,Tuf","Bas-a,Tp0-a,Cl/G-b,Dns-d","Dns-a,Bas-a,Tp0-a,Bas-b,Dns-d,Tuf","Dns-a,Bas-a,Tp0-a,Bas-b,Dns-d,Tuf","Dns-a,Bas-a,Tp0-a,Dns-d,Tuf","Tp0-a,Bas-b,Dns-d","Bas-a,Tp0-a,Dns-d,Tuf",null,"Dns-c","Tp0-a","Dns-a,Tp0-a,Bas-b,Prl-b,Dns-d,Tuf","Dns-a,Tp0-a,Prl-b,Dns-d,Tuf","Tp0-a,Prl-b,Cl/G-b,Dns-d",null,"Tp0-a","Dns-a,Tp0-a,Bas-b,Prl-b,Dns-d,Tuf","Dns-a,Tp0-a,Bas-b,Prl-b,Dns-d,Tuf","Tp0-a,Prl-b,Cl/G-b","Tp0-a,Prl-b,Dns-d,Tuf",null,null,null,null,"Dns-a,Bas-a,Tp0-a,Prl-b,Dns-d,Tuf","Dns-a,Dns-d","Tp0-a,Prl-b,Dns-d","Dns-a,Bas-a,Tp0-a,Prl-b,Dns-d,Tuf","Dns-a,Bas-a,Tp0-a,Prl-b,Dns-d,Tuf",null,null,null,null,null,null,null,null,"Tp0-b,Tuf",null,"Cl/G-b",null,"Hil-b,Tuf",null,null,null,null,null,null,null,null,null,null,"Bas-a,Tp0-a,Prl-b,Dns-d","Bas-a,Tp0-a,Prl-b","Cil-c","Bas-d",null,null,null,null,null,null,null,null,null,null,null,"Bas-e,Tuf","Tuf","Tuf",null,null,null,null,null,"Bas-a,Tp0-a,Prl-b,Dns-d","Dns-a,Bas-a,Tp0-a,Bas-b,Prl-b,Dns-d","Dns-a,Bas-a,Tp0-a,Prl-b,Dns-d","Bas-e,Hil-b",null,"Hil-b","Dns-a,Bas-a,Tp0-a,Bas-b,Prl-b,Dns-d","Dns-a,Bas-a,Tp0-a,Bas-b,Prl-b,Dns-d",null,"Hil-b",null,null,null,"Bas-c,Tp0-b",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"Dns-a,Bas-a,Bas-b,Prl-b,Dns-d,Tuf","Dns-a,Bas-a,Prl-b,Dns-d,Tuf","Bas-a,Prl-b,Cl/G-b,Dns-d",null,"Dns-a,Bas-a,Bas-b,Prl-b,Dns-d,Tuf","Dns-a,Bas-a,Bas-b,Prl-b,Dns-d,Tuf","Bas-a,Prl-b,Dns-d,Tuf","Bas-a,Prl-b,Cl/G-b","Bas-b,Prl-b,Dns-d","Bas-a","Bas-a",null,"Bas-c,Tuf",null,null,null,null,null,null,null,null,"Dns-d",null,"Bas-a,Tp0-a,Bas-b,Prl-b,Dns-d,Tuf","Bas-a,Tp0-a,Prl-b,Dns-d,Tuf",null,null,"Bas-b,Dns-d","Bas-a,Tp0-a,Bas-b,Prl-b,Dns-d,Tuf","Bas-a,Tp0-a,Bas-b,Prl-b,Dns-d,Tuf","Dns-d",null,null,null,null,"Prl-c",null,null,"Dns-a,Bas-a,Tp0-a,Bas-b,Prl-b,Tuf","Dns-a,Bas-a,Tp0-a,Prl-b,Tuf",null,null,null,"Dns-a,Bas-a,Tp0-a,Bas-b,Prl-b,Tuf","Dns-a,Bas-a,Tp0-a,Bas-b,Prl-b,Tuf","Bas-a,Tp0-a,Prl-b,Tuf","Tp0-a,Bas-b,Prl-b","Dns-a,Bas-b","Bas-a,Tp0-a,Prl-b,Cl/G-b","Dns-a","Dns-a"]},"columns":[{"id":"label","name":"Component","type":"factor"},{"id":"geneSet","name":"Gene Set","type":"character"},{"id":"description","name":"Description","type":"character"},{"id":"FDR","name":"FDR","type":"character"},{"id":"shared_by","name":"Shared By","type":"character"}],"groupBy":["label"],"dataKey":"953c888262a8c45f4ed60f3e0b9fb653"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="topic-model-annotations" class="section level2">
<h2>Topic model annotations</h2>
<pre class="r"><code>tm_GSEA &lt;- readRDS(&quot;output/montoro-tm-gsea.rds&quot;)
make_tbl(tm_GSEA, tm_ksort[c(16:30, 6:15, 1:5)])</code></pre>
<div class="reactable html-widget html-fill-item" id="htmlwidget-0c6e50d58f9aaf7a4d3a" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-0c6e50d58f9aaf7a4d3a">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"label":["Cil","Prl","Prl","Prl","Prl","Prl","B/P-a","B/P-b","B/P-b","B/P-b","B/P-b","Bas","Cl/G","Cl/Hil","Cl/Hil","Cl/Hil","Cl/Hil","Club","Club","H/G-a","H/G-a","H/G-a","H/G-b","Hil","Gob1","Gob1","Gob1","Gob1","Gob1","Gob1","Gob1","Gob1","Gob1","Gob1","Gob2","Gob2","Gob2","Gob2","Gob2","Gob2","Gob2","Gob2","Gob2","TNI-a","TNI-a","TNI-a","TNI-a","TNI-a","TNI-b","TpxCl","Tp0/60-a","Tp0/60-b","Tp30/60","Tp0-a","Tp0-b","Tp0-b","Tp0-b","Tp0-b","Tp0-b","Tp0-b","Tp0-b","Tp0-b","Tp0-b","Tp30xCl","Tp30xCl","Tp30xCl","Tp30xCl","Tp30xCl","Tp30-a","Tp30-a","Tp30-a","Tp30-a","Tp30-b","Tp60","Tp60","Tp60","Dns-a","Dns-b","Dns-c","Dns-c","Dns-c","Dns-d","Dns-d","Dns-d","Dns-d","Dns-d","Dns-d","Dns-d","Dns-d","Dns-d","Dns-e"],"geneSet":["GO:0030534","GO:0007368","GO:0004888","GO:0005901","GO:0008203","GO:1902652","GO:0071229","GO:0008203","GO:1902652","GO:0016125","GO:0030183","GO:0035869","GO:0042802","GO:0005840","GO:0022626","GO:0044391","GO:0003735","GO:0005743","GO:0019866","GO:0007154","GO:0007165","GO:0023052","GO:0046483","GO:0008654","GO:0043604","GO:0044445","GO:0022626","GO:0043232","GO:0043228","GO:1901566","GO:0043043","GO:1990904","GO:0005840","GO:0006412","GO:0006396","GO:0044445","GO:0044391","GO:0005840","GO:0022626","GO:0003735","GO:0015935","GO:0002181","GO:1990904","GO:0007259","GO:0097696","GO:0019226","GO:0008138","GO:0071526","GO:0097305","GO:0002526","GO:0007369","GO:0001508","GO:0014902","GO:0007224","GO:0001678","GO:0071322","GO:0071333","GO:0071331","GO:0071326","GO:0033002","GO:0007264","GO:0048660","GO:0048659","GO:0046578","GO:0051056","GO:0032609","GO:0032649","GO:0045580","GO:0072330","GO:1903076","GO:1904375","GO:0019003","GO:0004672","GO:0016614","GO:0016616","GO:0002028","GO:0022900","GO:0001664","GO:0061138","GO:0001763","GO:0048754","GO:0005794","GO:0005525","GO:0019001","GO:0032561","GO:0001883","GO:0032550","GO:0032549","GO:0045934","GO:0001882","GO:0022613"],"description":["adult behavior","determination of left/right symmetry","transmembrane signaling receptor activity","caveola","cholesterol metabolic process","secondary alcohol metabolic process","cellular response to acid chemical","cholesterol metabolic process","secondary alcohol metabolic process","sterol metabolic process","B cell differentiation","ciliary transition zone","identical protein binding","ribosome","cytosolic ribosome","ribosomal subunit","structural constituent of ribosome","mitochondrial inner membrane","organelle inner membrane","cell communication","signal transduction","signaling","heterocycle metabolic process","phospholipid biosynthetic process","amide biosynthetic process","cytosolic part","cytosolic ribosome","intracellular non-membrane-bounded organelle","non-membrane-bounded organelle","organonitrogen compound biosynthetic process","peptide biosynthetic process","ribonucleoprotein complex","ribosome","translation","RNA processing","cytosolic part","ribosomal subunit","ribosome","cytosolic ribosome","structural constituent of ribosome","small ribosomal subunit","cytoplasmic translation","ribonucleoprotein complex","JAK-STAT cascade","STAT cascade","transmission of nerve impulse","protein tyrosine/serine/threonine phosphatase activity","semaphorin-plexin signaling pathway","response to alcohol","acute inflammatory response","gastrulation","action potential","myotube differentiation","smoothened signaling pathway","cellular glucose homeostasis","cellular response to carbohydrate stimulus","cellular response to glucose stimulus","cellular response to hexose stimulus","cellular response to monosaccharide stimulus","muscle cell proliferation","small GTPase mediated signal transduction","regulation of smooth muscle cell proliferation","smooth muscle cell proliferation","regulation of Ras protein signal transduction","regulation of small GTPase mediated signal transduction","interferon-gamma production","regulation of interferon-gamma production","regulation of T cell differentiation","monocarboxylic acid biosynthetic process","regulation of protein localization to plasma membrane","regulation of protein localization to cell periphery","GDP binding","protein kinase activity","oxidoreductase activity, acting on CH-OH group of donors","oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor","regulation of sodium ion transport","electron transport chain","G protein-coupled receptor binding","morphogenesis of a branching epithelium","morphogenesis of a branching structure","branching morphogenesis of an epithelial tube","Golgi apparatus","GTP binding","guanyl nucleotide binding","guanyl ribonucleotide binding","purine nucleoside binding","purine ribonucleoside binding","ribonucleoside binding","negative regulation of nucleobase-containing compound metabolic process","nucleoside binding","ribonucleoprotein complex biogenesis"],"FDR":["<.01","<.01","<.01","0.05","0.05","0.05","<.01","<.01","<.01","<.01","0.02","0.03","0.25","<.01","<.01","<.01","<.01","0.27","0.27","0.33","0.33","0.33","0.49","0.11","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","0.01","0.03","0.03","<.01","<.01","<.01","0.01","0.02","0.21","0.07","0.18","0.44","0.02","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","<.01","0.02","0.02","0.04","0.03","0.03","0.04","0.05","0.14","<.01","<.01","0.02","0.05","0.22","<.01","<.01","<.01","0.02","0.02","0.02","0.02","0.02","0.02","0.02","0.03","0.05","0.20"],"shared_by":[null,null,null,null,"B/P-b","B/P-b",null,"Prl","Prl",null,null,null,null,"Gob2,Gob1","Gob2,Gob1","Gob2","Gob2",null,null,null,null,null,null,null,null,"Gob2","Gob2,Cl/Hil",null,null,null,null,"Gob2","Gob2,Cl/Hil",null,null,"Gob1","Cl/Hil","Gob1,Cl/Hil","Gob1,Cl/Hil","Cl/Hil",null,null,"Gob1",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null]},"columns":[{"id":"label","name":"Component","type":"factor"},{"id":"geneSet","name":"Gene Set","type":"character"},{"id":"description","name":"Description","type":"character"},{"id":"FDR","name":"FDR","type":"character"},{"id":"shared_by","name":"Shared By","type":"character"}],"groupBy":["label"],"dataKey":"d0ccfec855a3f63c1b4e7cfc245bde75"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.1 (2024-06-14)
Platform: aarch64-apple-darwin20
Running under: macOS Sonoma 14.7.1

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: America/New_York
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] reactable_0.4.4 cowplot_1.1.3   lubridate_1.9.3 forcats_1.0.0  
 [5] stringr_1.5.1   dplyr_1.1.4     purrr_1.0.2     readr_2.1.5    
 [9] tidyr_1.3.1     tibble_3.2.1    ggplot2_3.5.1   tidyverse_2.0.0
[13] workflowr_1.7.1

loaded via a namespace (and not attached):
 [1] tidyselect_1.2.1    viridisLite_0.4.2   farver_2.1.2       
 [4] fastmap_1.2.0       lazyeval_0.2.2      promises_1.3.2     
 [7] digest_0.6.37       timechange_0.3.0    lifecycle_1.0.4    
[10] processx_3.8.4      invgamma_1.1        magrittr_2.0.3     
[13] compiler_4.4.1      rlang_1.1.5         sass_0.4.9         
[16] progress_1.2.3      tools_4.4.1         yaml_2.3.10        
[19] data.table_1.16.2   knitr_1.49          prettyunits_1.2.0  
[22] labeling_0.4.3      htmlwidgets_1.6.4   Rtsne_0.17         
[25] withr_3.0.2         grid_4.4.1          git2r_0.33.0       
[28] fastTopics_0.7-07   colorspace_2.1-1    scales_1.3.0       
[31] gtools_3.9.5        cli_3.6.3           rmarkdown_2.29     
[34] crayon_1.5.3        generics_0.1.3      RcppParallel_5.1.9 
[37] rstudioapi_0.17.0   httr_1.4.7          tzdb_0.4.0         
[40] pbapply_1.7-2       cachem_1.1.0        parallel_4.4.1     
[43] vctrs_0.6.5         Matrix_1.7-0        jsonlite_1.8.9     
[46] callr_3.7.6         hms_1.1.3           mixsqp_0.3-54      
[49] ggrepel_0.9.6       irlba_2.3.5.1       crosstalk_1.2.1    
[52] plotly_4.10.4       jquerylib_0.1.4     glue_1.8.0         
[55] reactR_0.6.1        ps_1.8.0            uwot_0.2.2         
[58] stringi_1.8.4       gtable_0.3.6        later_1.4.1        
[61] quadprog_1.5-8      munsell_0.5.1       pillar_1.10.1      
[64] htmltools_0.5.8.1   truncnorm_1.0-9     R6_2.5.1           
[67] rprojroot_2.0.4     evaluate_1.0.3      lattice_0.22-6     
[70] RhpcBLASctl_0.23-42 SQUAREM_2021.1      ashr_2.2-63        
[73] httpuv_1.6.15       bslib_0.9.0         Rcpp_1.0.14        
[76] whisker_0.4.1       xfun_0.50           fs_1.6.5           
[79] getPass_0.2-4       pkgconfig_2.0.3    </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
