<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Jason Willwerscheid" />


<title>Montoro et al. structure plots</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">ebmf-for-scrnaseq-paper</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">Montoro et al. structure plots</h1>
<h4 class="author">Jason Willwerscheid</h4>
<h4 class="date">2/2/2025</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-02-05
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ebmf-for-scrnaseq-paper/</code>
<span class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220420code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20220420)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220420code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220420)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomwillwerscheidebmfforscrnaseqpapertree87d10f4264c45213eafecde52fc195a3312ed8dftargetblank87d10f4a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/willwerscheid/ebmf-for-scrnaseq-paper/tree/87d10f4264c45213eafecde52fc195a3312ed8df" target="_blank">87d10f4</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomwillwerscheidebmfforscrnaseqpapertree87d10f4264c45213eafecde52fc195a3312ed8dftargetblank87d10f4a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/willwerscheid/ebmf-for-scrnaseq-paper/tree/87d10f4264c45213eafecde52fc195a3312ed8df" target="_blank">87d10f4</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/.DS_Store
    Ignored:    data/montoro-mean-expr.rds
    Ignored:    data/montoro.rds
    Ignored:    data/pijuan-sala-mean-expr.rds
    Ignored:    data/pijuan-sala.rds
    Ignored:    figs/
    Ignored:    output/montoro-ebmf.rds
    Ignored:    output/montoro-ebnmf-gsea.rds
    Ignored:    output/montoro-nmf-gsea.rds
    Ignored:    output/montoro-nmf.rds
    Ignored:    output/montoro-tm-gsea.rds
    Ignored:    output/montoro-topics.rds
    Ignored:    output/ps-ebmf-20.rds
    Ignored:    output/ps-ebmf-30.rds
    Ignored:    output/ps-ebmf-40.rds
    Ignored:    output/ps-nmf.rds
    Ignored:    output/ps-topics-20.rds
    Ignored:    output/ps-topics-30.rds
    Ignored:    output/ps-topics-40.rds

Untracked files:
    Untracked:  code/montoro_gsea.R
    Untracked:  code/montoro_gseasusie2.R
    Untracked:  output/ebnmf_annotations.html

Unstaged changes:
    Deleted:    analysis/montoro_structure.html
    Modified:   analysis/montoro_structure.qmd
    Deleted:    analysis/montoro_structure_files/figure-html/unnamed-chunk-10-1.png
    Deleted:    analysis/montoro_structure_files/figure-html/unnamed-chunk-11-1.png
    Deleted:    analysis/montoro_structure_files/figure-html/unnamed-chunk-11-2.png
    Deleted:    analysis/montoro_structure_files/figure-html/unnamed-chunk-11-3.png
    Deleted:    analysis/montoro_structure_files/figure-html/unnamed-chunk-12-1.png
    Deleted:    analysis/montoro_structure_files/figure-html/unnamed-chunk-12-2.png
    Deleted:    analysis/montoro_structure_files/figure-html/unnamed-chunk-12-3.png
    Deleted:    analysis/montoro_structure_files/figure-html/unnamed-chunk-5-1.png
    Deleted:    analysis/montoro_structure_files/figure-html/unnamed-chunk-6-1.png
    Deleted:    analysis/montoro_structure_files/figure-html/unnamed-chunk-7-1.png
    Deleted:    analysis/montoro_structure_files/figure-html/unnamed-chunk-8-1.png
    Deleted:    analysis/montoro_structure_files/figure-html/unnamed-chunk-9-1.png
    Deleted:    analysis/montoro_structure_files/libs/bootstrap/bootstrap-icons.css
    Deleted:    analysis/montoro_structure_files/libs/bootstrap/bootstrap-icons.woff
    Deleted:    analysis/montoro_structure_files/libs/bootstrap/bootstrap.min.css
    Deleted:    analysis/montoro_structure_files/libs/bootstrap/bootstrap.min.js
    Deleted:    analysis/montoro_structure_files/libs/clipboard/clipboard.min.js
    Deleted:    analysis/montoro_structure_files/libs/quarto-html/anchor.min.js
    Deleted:    analysis/montoro_structure_files/libs/quarto-html/popper.min.js
    Deleted:    analysis/montoro_structure_files/libs/quarto-html/quarto-syntax-highlighting.css
    Deleted:    analysis/montoro_structure_files/libs/quarto-html/quarto.js
    Deleted:    analysis/montoro_structure_files/libs/quarto-html/tippy.css
    Deleted:    analysis/montoro_structure_files/libs/quarto-html/tippy.umd.min.js

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/montoro_structure.Rmd</code>) and
HTML (<code>docs/montoro_structure.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/willwerscheid/ebmf-for-scrnaseq-paper/blob/87d10f4264c45213eafecde52fc195a3312ed8df/analysis/montoro_structure.Rmd" target="_blank">87d10f4</a>
</td>
<td>
Jason Willwerscheid
</td>
<td>
2025-02-05
</td>
<td>
wflow_publish("analysis/montoro_structure.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/willwerscheid/ebmf-for-scrnaseq-paper/5b161280729dfb4707f4e0eb9ba65888e8391103/docs/montoro_structure.html" target="_blank">5b16128</a>
</td>
<td>
Jason Willwerscheid
</td>
<td>
2025-02-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/willwerscheid/ebmf-for-scrnaseq-paper/blob/61633b16feebc600834b1a4635cb305d749cae31/analysis/montoro_structure.Rmd" target="_blank">61633b1</a>
</td>
<td>
Jason Willwerscheid
</td>
<td>
2025-02-02
</td>
<td>
workflowr::wflow_publish("analysis/montoro_structure.Rmd")
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="introduction" class="section level2">
<h2>Introduction</h2>
<p>Here I produce structure plots for EBNMF, NMF, and topic model fits
to the larger dataset from Montoro et al.</p>
<p>Load the Tidyverse:</p>
<pre class="r"><code>library(tidyverse)
library(cowplot)
library(gt)</code></pre>
<p>Load the <code>flashier</code> results:</p>
<pre class="r"><code>scale_FF &lt;- function(FF, LL, D = 1) {
  LL_norms &lt;- apply(LL, 2, function(x) sqrt(sum(x^2)))
  return(t(t(FF) * D * LL_norms))
}

ebnmf &lt;- readRDS(&quot;output/montoro-ebmf.rds&quot;)
ebnmf_FF &lt;- scale_FF(ebnmf$fit$F.pm, ebnmf$fit$L.pm)</code></pre>
<p>Extract cell covariates:</p>
<pre class="r"><code>cell_types &lt;- c(
      &quot;Ciliated&quot;,
      &quot;Proliferating&quot;,
      &quot;Basal&quot;,
      &quot;Club&quot;,
      &quot;Club (hillock-associated)&quot;,
      &quot;Goblet.progenitor&quot;,
      &quot;Goblet.1&quot;,
      &quot;Goblet.2&quot;,
      &quot;Tuft.progenitor&quot;,
      &quot;Tuft.1&quot;,
      &quot;Tuft.2&quot;,
      &quot;Neuroendocrine&quot;,
      &quot;Ionocyte&quot;
)

ct_abbr &lt;- cell_types
ct_abbr &lt;- str_replace(ct_abbr, &quot;\\.&quot;, &quot;\\-&quot;)
ct_abbr[2] &lt;- &quot;Prolif.&quot;
ct_abbr[5] &lt;- &quot;Hillock&quot;
ct_abbr[12] &lt;- &quot;NEC&quot;
ct_abbr &lt;- str_replace(ct_abbr, &quot;progenitor&quot;, &quot;Pr.&quot;)

external_info &lt;- tibble(
  CellType = sapply(strsplit(rownames(ebnmf_FF), &quot;_&quot;), `[[`, 5),
  TimePoint = sapply(strsplit(rownames(ebnmf_FF), &quot;_&quot;), `[[`, 2),
  Replicate = sapply(strsplit(rownames(ebnmf_FF), &quot;_&quot;), `[[`, 3)
) |&gt;
  left_join(tibble(CellType = cell_types, CtAbbr = ct_abbr), by = &quot;CellType&quot;) |&gt;
  mutate(
    CellType = factor(CellType, levels = cell_types),
    CtAbbr = factor(CtAbbr, levels = ct_abbr),
    TimePoint = factor(TimePoint),
    Replicate = factor(Replicate),
  ) |&gt;
  mutate(
    TPxRep = fct_cross(Replicate, TimePoint),
    CommonType = fct_collapse(CtAbbr, Other = ct_abbr[6:13])
  )</code></pre>
<p>I will use the following functions to generate plots:</p>
<pre class="r"><code>make_tib &lt;- function(FF, info_col, ksort, ncells = 50, duprate = 1) {
  info &lt;- external_info[[info_col]]

  # Downsample the number of cells.
  set.seed(666)
  cell_idx &lt;- numeric(0)
  for (k in levels(info)) {
    which_idx &lt;- which(info == k)
    # Downsample common cell types. Duplicate rare ones.
    if (ncells &lt; length(which_idx)) {
      which_idx &lt;- sample(which_idx, ncells, replace = FALSE)
    } else if (ncells &gt;= length(which_idx) * duprate) {
      which_idx &lt;- rep(which_idx, duprate)
    }
    cell_idx &lt;- c(cell_idx, which_idx)
  }

  FF &lt;- FF[cell_idx, ]
  info &lt;- info[cell_idx]
  colnames(FF) &lt;- 1:ncol(FF)

  lvls &lt;- as.numeric(ksort)
  if (any(setdiff(1:ncol(FF), ksort))) {
    lvls &lt;- c(&quot;Other&quot;, lvls)
  }

  tib &lt;- as_tibble(FF) |&gt;
    mutate(
      CellIdx = row_number(),
      Info = info
    ) |&gt;
    pivot_longer(
      -c(CellIdx, Info),
      names_to = &quot;Component&quot;,
      values_to = &quot;Loading&quot;,
      values_drop_na = TRUE
    ) |&gt;
    mutate(
      Component = ifelse(as.numeric(Component) %in% ksort, Component, &quot;Other&quot;)
    ) |&gt;
    group_by(CellIdx, Info, Component) |&gt;
    summarize(Loading = sum(Loading), .groups = &quot;drop&quot;) |&gt;
    mutate(
      Component = factor(Component, levels = lvls)
    ) |&gt;
    arrange(Info, Component)
  
  levels(tib$Component) &lt;- c(&quot;Other&quot;, names(ksort))

  return(tib)
}

do_plot &lt;- function(tib, colors, ptitle, ntiles = 400) {
  struct_df &lt;- tib |&gt;
    pivot_wider(names_from = Component, values_from = Loading)
  Lmat &lt;- struct_df |&gt;
    select(-(1:2)) |&gt;
    as.matrix()
  group &lt;- struct_df$Info
  
  if (&quot;Other&quot; %in% tib$Component) {
    colors &lt;- c(&quot;grey90&quot;, colors)
  }
  
  p &lt;- fastTopics::structure_plot(
    Lmat, topics = colnames(Lmat),
    grouping = group,
    colors = colors,
    # embed_method = fastTopics::umap_from_topics,
    # dims = 1,
    gap = 10,
    verbose = FALSE
  ) + theme(
    axis.text.y = element_blank()
  ) + labs(
    y = &quot;&quot;,
    title = ptitle
  )

  return(p)
}

glasbey &lt;- function(k) {
  return(fastTopics:::glasbey()[k])
}</code></pre>
<p>The following function is used to generate tables of gene set
annotations:</p>
<pre class="r"><code>make_tbl &lt;- function(GSEA, ksort) {
  klabs &lt;- tibble(
    korig = sort(ksort),
    ksort = ksort,
    label = factor(names(ksort), levels = names(ksort))
  )

  keep_res &lt;- GSEA |&gt;
    # Remove &quot;identical&quot; gene sets (userId lists genes found in set)
    left_join(klabs, by = c(&quot;k&quot; = &quot;ksort&quot;)) |&gt;
    group_by(k, userId) |&gt;
    arrange(FDR) |&gt;
    slice_head(n = 1) |&gt;
    group_by(k) |&gt;
    slice_min(FDR, n = 5) |&gt;
    filter(FDR &lt; 0.05 | rank(FDR) == min(rank(FDR))) |&gt;
    ungroup()

  shared_genesets &lt;- keep_res |&gt;
    group_by(geneSet) |&gt;
    filter(n() &gt; 1) |&gt;
    ungroup()
  distinctive_genesets &lt;- keep_res |&gt;
    anti_join(shared_genesets)

  shared_genesets &lt;- shared_genesets |&gt;
    group_by(geneSet, description) |&gt;
    arrange(label) |&gt;
    summarize(components = paste0(label, collapse = &quot;, &quot;),
              FDR = ifelse(
                n() &lt; 4,
                paste(ifelse(FDR &lt; .01, &quot;&lt;.01&quot;, round(FDR, 2)), collapse = &quot;, &quot;),
                paste(&quot;all &lt;&quot;, round(max(FDR) + 0.005001, 2))
              ),
              order1 = label[1], order2 = label[2],
              .groups = &quot;drop&quot;) |&gt;
    arrange(order1, order2) |&gt;
    select(-order1, -order2)
  distinctive_genesets &lt;- distinctive_genesets |&gt;
    group_by(geneSet, description) |&gt;
    summarize(components = paste0(label, collapse = &quot;, &quot;),
              order2 = FDR[1],
              FDR = paste(ifelse(FDR &lt; .01, &quot;&lt;.01&quot;, round(FDR, 2)), collapse = &quot;, &quot;),
              order1 = label[1],
              .groups = &quot;drop&quot;) |&gt;
    arrange(order1, order2) |&gt;
    select(-order1, -order2)

  genesets &lt;- shared_genesets |&gt;
    bind_rows(distinctive_genesets) |&gt;
    group_by(components)

  gt_tbl &lt;- gt(genesets) |&gt;
    #cols_move_to_start(columns = components) |&gt;
    cols_label(
      geneSet = &quot;Gene Set&quot;,
      description = &quot;Description&quot;,
      components = &quot;Components&quot;
    ) |&gt;
    tab_style(
      style = cell_text(weight = &quot;bold&quot;),
      locations = cells_row_groups()
    ) |&gt;
    tab_options(
      table.font.size = 11
    )
  return(gt_tbl)
}</code></pre>
</div>
<div id="ebnmf" class="section level2">
<h2>EBNMF</h2>
<p>In each case I manually sort and label factors.</p>
<pre class="r"><code>ebnmf_ksort &lt;- c(
  26, 27, 30, # Sparse
  1, 8, 10, 16, # Dense
  11, 12, 2, 24, # Time points
  6, 13, 22, # Ciliated
  21, 23, 17, # Proliferating
  4, 5, 9, # Basal
  3, 15, 7, # 26, Club-Hillock
  25, 28, 29, # Goblet
  18, 19, 14, 20 # Tuft-NEC-Ionocyte
)
names(ebnmf_ksort) &lt;- c(
  &quot;Sprs-a&quot;, &quot;Sprs-b&quot;, &quot;Sprs-c&quot;,
  &quot;Dns-a&quot;, &quot;Dns-b&quot;, &quot;Dns-c&quot;, &quot;Dns-d&quot;,
  &quot;Tp0&quot;, &quot;Tp30&quot;, &quot;Tp60&quot;, &quot;R1xTp30&quot;,
  &quot;Cil-a&quot;, &quot;Cil-b&quot;, &quot;Cil-c&quot;,
  &quot;Prl-a&quot;, &quot;Prl-b&quot;, &quot;Prl-c&quot;,
  &quot;Bas-a&quot;, &quot;Bas-b&quot;, &quot;Bas-c&quot;,
  &quot;Cl/G-a&quot;, &quot;Cl/G-b&quot;, &quot;Hil&quot;,
  &quot;Gob&quot;, &quot;Gob1&quot;, &quot;Gob2&quot;,
  &quot;Tuf1&quot;, &quot;Tuf2&quot;, &quot;Nec&quot;, &quot;Ion&quot;
)

sparse_title &lt;- &quot;Very sparse factors&quot;
dense_title &lt;- &quot;Dense factors&quot;
tp_title &lt;- &quot;Time point factors&quot;
common_title &lt;- &quot;Prevalent cell type factors&quot;
rare_title &lt;- &quot;Rare cell type factors (common cell types are downsampled)&quot;

sparse_k &lt;- 1:3
ebnmf_tib &lt;- make_tib(ebnmf_FF, &quot;CommonType&quot;, ebnmf_ksort[sparse_k], Inf, 1)
sparse_p &lt;- do_plot(ebnmf_tib, glasbey(2:4), sparse_title)

dense_k &lt;- 4:7
ebnmf_tib &lt;- make_tib(ebnmf_FF, &quot;CommonType&quot;, ebnmf_ksort[dense_k], Inf, 1)
dense_p &lt;- do_plot(ebnmf_tib, glasbey(5:8), dense_title)

tp_k &lt;- 8:11
ebnmf_tib &lt;- make_tib(ebnmf_FF, &quot;TPxRep&quot;, ebnmf_ksort[tp_k], Inf, 1)
tp_p &lt;- do_plot(ebnmf_tib, glasbey(14:17), tp_title)

common_k &lt;- c(12:14, 18:20, 23)
ebnmf_tib &lt;- make_tib(ebnmf_FF, &quot;CommonType&quot;, ebnmf_ksort[common_k], Inf, 1)
common_p &lt;- do_plot(ebnmf_tib, glasbey(21:27), common_title)

rare_k &lt;- c(15:17, 21:22, 24:30)
ebnmf_tib &lt;- make_tib(ebnmf_FF, &quot;CtAbbr&quot;, ebnmf_ksort[rare_k], 100, 3)
rare_p &lt;- do_plot(ebnmf_tib, glasbey(32:43), rare_title)

plot_grid(common_p, rare_p, tp_p, dense_p, sparse_p,  nrow = 5, ncol = 1)</code></pre>
<p><img src="figure/montoro_structure.Rmd/unnamed-chunk-6-1.png" width="768" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-6-1">
Past versions of unnamed-chunk-6-1.png
</button>
</p>
<div id="fig-unnamed-chunk-6-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/willwerscheid/ebmf-for-scrnaseq-paper/blob/5b161280729dfb4707f4e0eb9ba65888e8391103/docs/figure/montoro_structure.Rmd/unnamed-chunk-6-1.png" target="_blank">5b16128</a>
</td>
<td>
Jason Willwerscheid
</td>
<td>
2025-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="nmf" class="section level2">
<h2>NMF</h2>
<pre class="r"><code>nnlm &lt;- readRDS(&quot;output/montoro-nmf.rds&quot;)
nnlm_FF &lt;- scale_FF(t(nnlm$fit$h), nnlm$fit$w, nnlm$fit$d)

nnlm_ksort &lt;- c(
  1, 28, 10, 26, # dense
  9, 5, # time points (dense)
  3, 11, 8, 7, # time points
  21, # replicate x TP
  15, 22, 20, # ciliated
  24, 12, 19, # prolif
  2, 6, 4, 14, 13, # basal
  25, 17, 23, # club/goblet
  18, 16, # hillock
  29, # tuft
  27, # NEC
  30 # ionocyte
)
names(nnlm_ksort) &lt;- c(
  &quot;Dns-a&quot;, &quot;Dns-b&quot;, &quot;Dns-c&quot;, &quot;Dns-d&quot;,
  &quot;TpxCl&quot;, &quot;Tp30/60&quot;,
  &quot;Tp0-a&quot;, &quot;Tp0-b&quot;, &quot;Tp30&quot;, &quot;Tp60&quot;,
  &quot;R1xTp30&quot;,
  &quot;Cil-a&quot;, &quot;Cil-b&quot;, &quot;Cil-c&quot;,  
  &quot;Prl-a&quot;, &quot;Prl-b&quot;, &quot;Prl-c&quot;,
  &quot;Bas-a&quot;, &quot;Bas-b&quot;, &quot;Bas-c&quot;, &quot;Bas-d&quot;, &quot;Bas-e&quot;,
  &quot;Cl/G-a&quot;, &quot;Cl/G-b&quot;, &quot;Cl/G-c&quot;,
  &quot;Hil-a&quot;, &quot;Hil-b&quot;,
  &quot;Tuf&quot;,
  &quot;Nec&quot;,
  &quot;Ion/Cl/G&quot;
)

nnlm_dense_k &lt;- 1:4
nnlm_tib &lt;- make_tib(nnlm_FF, &quot;CommonType&quot;, nnlm_ksort[nnlm_dense_k], Inf, 1)
nnlm_dense_p &lt;- do_plot(nnlm_tib, glasbey(5:8), dense_title)

nnlm_tp_k &lt;- 5:11
nnlm_tib &lt;- make_tib(nnlm_FF, &quot;TPxRep&quot;, nnlm_ksort[nnlm_tp_k], Inf, 1)
nnlm_tp_p &lt;- do_plot(nnlm_tib, glasbey(11:17), tp_title)

nnlm_common_k &lt;- c(12:14, 18:22, 26:27)
nnlm_tib &lt;- make_tib(nnlm_FF, &quot;CommonType&quot;, nnlm_ksort[nnlm_common_k], Inf, 1)
nnlm_common_p &lt;- do_plot(nnlm_tib, glasbey(21:30), common_title)

nnlm_rare_k &lt;- c(15:17, 23:25, 28:30)
nnlm_tib &lt;- make_tib(nnlm_FF, &quot;CtAbbr&quot;, nnlm_ksort[nnlm_rare_k], 100, 3)
nnlm_rare_p &lt;- do_plot(nnlm_tib, glasbey(32:40), rare_title)

plot_grid(nnlm_common_p, nnlm_rare_p, nnlm_tp_p, nnlm_dense_p, nrow = 4, ncol = 1)</code></pre>
<p><img src="figure/montoro_structure.Rmd/unnamed-chunk-7-1.png" width="768" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-7-1">
Past versions of unnamed-chunk-7-1.png
</button>
</p>
<div id="fig-unnamed-chunk-7-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/willwerscheid/ebmf-for-scrnaseq-paper/blob/5b161280729dfb4707f4e0eb9ba65888e8391103/docs/figure/montoro_structure.Rmd/unnamed-chunk-7-1.png" target="_blank">5b16128</a>
</td>
<td>
Jason Willwerscheid
</td>
<td>
2025-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="topic-model" class="section level2">
<h2>Topic model</h2>
<pre class="r"><code>tm &lt;- readRDS(&quot;output/montoro-topics.rds&quot;)
tm_fit &lt;- tm$fit
tm_fit$F &lt;- tm$fit$L
tm_fit$L &lt;- tm$fit$F
tm_fit$Fn &lt;- tm$fit$Ln
tm_fit$Ln &lt;- tm$fit$Fn
tm_fit$Fy &lt;- tm$fit$Ly
tm_fit$Ly &lt;- tm$fit$Fy
tm_fit &lt;- fastTopics::poisson2multinom(tm_fit)
tm_FF &lt;- tm_fit$L

tm_ksort &lt;- c(
  21, 5, 11, 17, 27, # Dense  
  19, # Tp30/60xCl
  15, 8, # Tp0/60?
  24, # Tp30/60
  14, 26, # Tp0
  23, 29, 9, # Tp30 - 23 also club
  13, # Tp60
  25, # Ciliated
  6, # Proliferating
  10, 4, # Basal/Prolif
  12, # Basal
  3, # Club/Goblet
  18, # Club/Hillock
  22, # Club
  16, 28, # Hillock/Goblet-progenitor
  20, # Hillock
  7, # Goblet-p/1
  2, # Goblet-2
  1, 30 # Tuft-NEC-Ion
)
names(tm_ksort) &lt;- c(
  &quot;Dns-a&quot;, &quot;Dns-b&quot;, &quot;Dns-c&quot;,
  &quot;Dns-d&quot;, &quot;Dns-e&quot;,
  &quot;TpxCl&quot;,
  &quot;Tp0/60-a&quot;, &quot;Tp0/60-b&quot;,
  &quot;Tp30/60&quot;,
  &quot;Tp0-a&quot;, &quot;Tp0-b&quot;,
  &quot;Tp30xCl&quot;, &quot;Tp30-a&quot;, &quot;Tp30-b&quot;,
  &quot;Tp60&quot;,
  &quot;Cil&quot;,
  &quot;Prl&quot;,
  &quot;B/P-a&quot;, &quot;B/P-b&quot;,
  &quot;Bas&quot;, 
  &quot;Cl/G&quot;, &quot;Cl/Hil&quot;, &quot;Club&quot;,
  &quot;H/G-a&quot;, &quot;H/G-b&quot;, 
  &quot;Hil&quot;, 
  &quot;Gob1&quot;, &quot;Gob2&quot;,
  &quot;TNI-a&quot;, &quot;TNI-b&quot;
)

tm_dense_k &lt;- 1:5
tm_tib &lt;- make_tib(tm_FF, &quot;CommonType&quot;, tm_ksort[tm_dense_k], Inf, 1)
tm_dense_p &lt;- do_plot(tm_tib, glasbey(5:9), dense_title)

tm_tp_k &lt;- 6:15
tm_tib &lt;- make_tib(tm_FF, &quot;TPxRep&quot;, tm_ksort[tm_tp_k], Inf, 1)
tm_tp_p &lt;- do_plot(tm_tib, glasbey(11:20), tp_title)

tm_common_k &lt;- c(16, 20, 22:23, 26)
tm_tib &lt;- make_tib(tm_FF, &quot;CommonType&quot;, tm_ksort[tm_common_k], Inf, 1)
tm_common_p &lt;- do_plot(tm_tib, glasbey(24:29), common_title)

tm_rare_k &lt;- c(17:19, 21, 24:25, 27:30)
tm_tib &lt;- make_tib(tm_FF, &quot;CtAbbr&quot;, tm_ksort[tm_rare_k], 100, 3)
tm_rare_p &lt;- do_plot(tm_tib, glasbey(32:41), rare_title)

plot_grid(tm_common_p, tm_rare_p, tm_tp_p, tm_dense_p, nrow = 4, ncol = 1)</code></pre>
<p><img src="figure/montoro_structure.Rmd/unnamed-chunk-8-1.png" width="768" style="display: block; margin: auto;" /></p>
</div>
<div id="ebnmf-annotations" class="section level2">
<h2>EBNMF annotations</h2>
<p>I use GSEA to do annotations. For each component, I provide as input
all genes with loadings that are larger than 0.01 times the maximum
loading (for the topic model, 0.001 times the maximum loading). (If all
loadings are provided, GSEA will also find gene sets with very small
loadings across genes, which is obviously not what we want.) I report
the top 5 gene sets with FDR &lt; 0.05 for each component (ties are
included), with the restriction that at least one gene set is reported
for each factor (even if its FDR is greater than 0.05).</p>
<pre class="r"><code>ebnmf_GSEA &lt;- readRDS(&quot;output/montoro-ebnmf-gsea.rds&quot;)
make_tbl(ebnmf_GSEA, ebnmf_ksort[c(12:30, 8:11, 4:7, 1:3)])</code></pre>
<div id="eusuubewdf" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#eusuubewdf table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#eusuubewdf thead, #eusuubewdf tbody, #eusuubewdf tfoot, #eusuubewdf tr, #eusuubewdf td, #eusuubewdf th {
  border-style: none;
}

#eusuubewdf p {
  margin: 0;
  padding: 0;
}

#eusuubewdf .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 11px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#eusuubewdf .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#eusuubewdf .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#eusuubewdf .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#eusuubewdf .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#eusuubewdf .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#eusuubewdf .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#eusuubewdf .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#eusuubewdf .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#eusuubewdf .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#eusuubewdf .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#eusuubewdf .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#eusuubewdf .gt_spanner_row {
  border-bottom-style: hidden;
}

#eusuubewdf .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#eusuubewdf .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#eusuubewdf .gt_from_md > :first-child {
  margin-top: 0;
}

#eusuubewdf .gt_from_md > :last-child {
  margin-bottom: 0;
}

#eusuubewdf .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#eusuubewdf .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#eusuubewdf .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#eusuubewdf .gt_row_group_first td {
  border-top-width: 2px;
}

#eusuubewdf .gt_row_group_first th {
  border-top-width: 2px;
}

#eusuubewdf .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#eusuubewdf .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#eusuubewdf .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#eusuubewdf .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#eusuubewdf .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#eusuubewdf .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#eusuubewdf .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#eusuubewdf .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#eusuubewdf .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#eusuubewdf .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#eusuubewdf .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#eusuubewdf .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#eusuubewdf .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#eusuubewdf .gt_left {
  text-align: left;
}

#eusuubewdf .gt_center {
  text-align: center;
}

#eusuubewdf .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#eusuubewdf .gt_font_normal {
  font-weight: normal;
}

#eusuubewdf .gt_font_bold {
  font-weight: bold;
}

#eusuubewdf .gt_font_italic {
  font-style: italic;
}

#eusuubewdf .gt_super {
  font-size: 65%;
}

#eusuubewdf .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#eusuubewdf .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#eusuubewdf .gt_indent_1 {
  text-indent: 5px;
}

#eusuubewdf .gt_indent_2 {
  text-indent: 10px;
}

#eusuubewdf .gt_indent_3 {
  text-indent: 15px;
}

#eusuubewdf .gt_indent_4 {
  text-indent: 20px;
}

#eusuubewdf .gt_indent_5 {
  text-indent: 25px;
}

#eusuubewdf .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#eusuubewdf div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>
<table class="gt_table" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
  <thead>
    <tr class="gt_col_headings">
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="geneSet">Gene Set</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="description">Description</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="FDR">FDR</th>
    </tr>
  </thead>
  <tbody class="gt_table_body">
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Prl-a, Prl-b">Prl-a, Prl-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Prl-a, Prl-b  geneSet" class="gt_row gt_left">GO:0007368</td>
<td headers="Prl-a, Prl-b  description" class="gt_row gt_left">determination of left/right symmetry</td>
<td headers="Prl-a, Prl-b  FDR" class="gt_row gt_left">0.01, &lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Prl-a, Bas-a, Tp30">Prl-a, Bas-a, Tp30</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Prl-a, Bas-a, Tp30  geneSet" class="gt_row gt_left">GO:0035869</td>
<td headers="Prl-a, Bas-a, Tp30  description" class="gt_row gt_left">ciliary transition zone</td>
<td headers="Prl-a, Bas-a, Tp30  FDR" class="gt_row gt_left">&lt;.01, 0.02, &lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Bas-a, Tp30">Bas-a, Tp30</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Bas-a, Tp30  geneSet" class="gt_row gt_left">GO:0097731</td>
<td headers="Bas-a, Tp30  description" class="gt_row gt_left">9+0 non-motile cilium</td>
<td headers="Bas-a, Tp30  FDR" class="gt_row gt_left">0.02, &lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Bas-a, Dns-c">Bas-a, Dns-c</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Bas-a, Dns-c  geneSet" class="gt_row gt_left">GO:0030990</td>
<td headers="Bas-a, Dns-c  description" class="gt_row gt_left">intraciliary transport particle</td>
<td headers="Bas-a, Dns-c  FDR" class="gt_row gt_left">0.02, &lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Bas-b, Tp30">Bas-b, Tp30</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Bas-b, Tp30  geneSet" class="gt_row gt_left">GO:0097730</td>
<td headers="Bas-b, Tp30  description" class="gt_row gt_left">non-motile cilium</td>
<td headers="Bas-b, Tp30  FDR" class="gt_row gt_left">0.03, &lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tuf1, Tuf2, Nec, Tp0, R1xTp30, Sprs-a">Tuf1, Tuf2, Nec, Tp0, R1xTp30, Sprs-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tuf1, Tuf2, Nec, Tp0, R1xTp30, Sprs-a  geneSet" class="gt_row gt_left">GO:0003735</td>
<td headers="Tuf1, Tuf2, Nec, Tp0, R1xTp30, Sprs-a  description" class="gt_row gt_left">structural constituent of ribosome</td>
<td headers="Tuf1, Tuf2, Nec, Tp0, R1xTp30, Sprs-a  FDR" class="gt_row gt_left">all &lt; 0.02</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tuf1, Tuf2, Nec, R1xTp30, Dns-a">Tuf1, Tuf2, Nec, R1xTp30, Dns-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tuf1, Tuf2, Nec, R1xTp30, Dns-a  geneSet" class="gt_row gt_left">GO:0005840</td>
<td headers="Tuf1, Tuf2, Nec, R1xTp30, Dns-a  description" class="gt_row gt_left">ribosome</td>
<td headers="Tuf1, Tuf2, Nec, R1xTp30, Dns-a  FDR" class="gt_row gt_left">all &lt; 0.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tuf1, Tuf2, Nec, Tp0, R1xTp30">Tuf1, Tuf2, Nec, Tp0, R1xTp30</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tuf1, Tuf2, Nec, Tp0, R1xTp30  geneSet" class="gt_row gt_left">GO:0022627</td>
<td headers="Tuf1, Tuf2, Nec, Tp0, R1xTp30  description" class="gt_row gt_left">cytosolic small ribosomal subunit</td>
<td headers="Tuf1, Tuf2, Nec, Tp0, R1xTp30  FDR" class="gt_row gt_left">all &lt; 0.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tuf1, Tuf2, Nec, Sprs-a">Tuf1, Tuf2, Nec, Sprs-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tuf1, Tuf2, Nec, Sprs-a  geneSet" class="gt_row gt_left">GO:0044445</td>
<td headers="Tuf1, Tuf2, Nec, Sprs-a  description" class="gt_row gt_left">cytosolic part</td>
<td headers="Tuf1, Tuf2, Nec, Sprs-a  FDR" class="gt_row gt_left">all &lt; 0.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tuf1, Tp0">Tuf1, Tp0</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tuf1, Tp0  geneSet" class="gt_row gt_left">GO:0044391</td>
<td headers="Tuf1, Tp0  description" class="gt_row gt_left">ribosomal subunit</td>
<td headers="Tuf1, Tp0  FDR" class="gt_row gt_left">&lt;.01, 0.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tuf2, Nec">Tuf2, Nec</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tuf2, Nec  geneSet" class="gt_row gt_left">GO:0022625</td>
<td headers="Tuf2, Nec  description" class="gt_row gt_left">cytosolic large ribosomal subunit</td>
<td headers="Tuf2, Nec  FDR" class="gt_row gt_left">&lt;.01, &lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Cil-a">Cil-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Cil-a  geneSet" class="gt_row gt_left">GO:0030534</td>
<td headers="Cil-a  description" class="gt_row gt_left">adult behavior</td>
<td headers="Cil-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Cil-a  geneSet" class="gt_row gt_left">GO:0060048</td>
<td headers="Cil-a  description" class="gt_row gt_left">cardiac muscle contraction</td>
<td headers="Cil-a  FDR" class="gt_row gt_left">0.04</td></tr>
    <tr><td headers="Cil-a  geneSet" class="gt_row gt_left">GO:0055117</td>
<td headers="Cil-a  description" class="gt_row gt_left">regulation of cardiac muscle contraction</td>
<td headers="Cil-a  FDR" class="gt_row gt_left">0.05</td></tr>
    <tr><td headers="Cil-a  geneSet" class="gt_row gt_left">GO:0009408</td>
<td headers="Cil-a  description" class="gt_row gt_left">response to heat</td>
<td headers="Cil-a  FDR" class="gt_row gt_left">0.05</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Cil-b">Cil-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Cil-b  geneSet" class="gt_row gt_left">GO:0007569</td>
<td headers="Cil-b  description" class="gt_row gt_left">cell aging</td>
<td headers="Cil-b  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Cil-c">Cil-c</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Cil-c  geneSet" class="gt_row gt_left">GO:0006090</td>
<td headers="Cil-c  description" class="gt_row gt_left">pyruvate metabolic process</td>
<td headers="Cil-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Cil-c  geneSet" class="gt_row gt_left">GO:0010712</td>
<td headers="Cil-c  description" class="gt_row gt_left">regulation of collagen metabolic process</td>
<td headers="Cil-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Cil-c  geneSet" class="gt_row gt_left">GO:0016614</td>
<td headers="Cil-c  description" class="gt_row gt_left">oxidoreductase activity, acting on CH-OH group of donors</td>
<td headers="Cil-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Cil-c  geneSet" class="gt_row gt_left">GO:0032922</td>
<td headers="Cil-c  description" class="gt_row gt_left">circadian regulation of gene expression</td>
<td headers="Cil-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Cil-c  geneSet" class="gt_row gt_left">GO:0032963</td>
<td headers="Cil-c  description" class="gt_row gt_left">collagen metabolic process</td>
<td headers="Cil-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Cil-c  geneSet" class="gt_row gt_left">GO:0032964</td>
<td headers="Cil-c  description" class="gt_row gt_left">collagen biosynthetic process</td>
<td headers="Cil-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Cil-c  geneSet" class="gt_row gt_left">GO:0046620</td>
<td headers="Cil-c  description" class="gt_row gt_left">regulation of organ growth</td>
<td headers="Cil-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Cil-c  geneSet" class="gt_row gt_left">GO:0060420</td>
<td headers="Cil-c  description" class="gt_row gt_left">regulation of heart growth</td>
<td headers="Cil-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Prl-a">Prl-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Prl-a  geneSet" class="gt_row gt_left">GO:0030879</td>
<td headers="Prl-a  description" class="gt_row gt_left">mammary gland development</td>
<td headers="Prl-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Prl-a  geneSet" class="gt_row gt_left">GO:0050954</td>
<td headers="Prl-a  description" class="gt_row gt_left">sensory perception of mechanical stimulus</td>
<td headers="Prl-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Prl-a  geneSet" class="gt_row gt_left">GO:0004930</td>
<td headers="Prl-a  description" class="gt_row gt_left">G protein-coupled receptor activity</td>
<td headers="Prl-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Prl-b">Prl-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Prl-b  geneSet" class="gt_row gt_left">GO:0009799</td>
<td headers="Prl-b  description" class="gt_row gt_left">specification of symmetry</td>
<td headers="Prl-b  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Prl-b  geneSet" class="gt_row gt_left">GO:0042393</td>
<td headers="Prl-b  description" class="gt_row gt_left">histone binding</td>
<td headers="Prl-b  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Prl-b  geneSet" class="gt_row gt_left">GO:0005930</td>
<td headers="Prl-b  description" class="gt_row gt_left">axoneme</td>
<td headers="Prl-b  FDR" class="gt_row gt_left">0.02</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Prl-c">Prl-c</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Prl-c  geneSet" class="gt_row gt_left">GO:0005882</td>
<td headers="Prl-c  description" class="gt_row gt_left">intermediate filament</td>
<td headers="Prl-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Bas-a">Bas-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Bas-a  geneSet" class="gt_row gt_left">GO:0016573</td>
<td headers="Bas-a  description" class="gt_row gt_left">histone acetylation</td>
<td headers="Bas-a  FDR" class="gt_row gt_left">0.02</td></tr>
    <tr><td headers="Bas-a  geneSet" class="gt_row gt_left">GO:0045844</td>
<td headers="Bas-a  description" class="gt_row gt_left">positive regulation of striated muscle tissue development</td>
<td headers="Bas-a  FDR" class="gt_row gt_left">0.03</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Bas-b">Bas-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Bas-b  geneSet" class="gt_row gt_left">GO:0008237</td>
<td headers="Bas-b  description" class="gt_row gt_left">metallopeptidase activity</td>
<td headers="Bas-b  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Bas-c">Bas-c</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Bas-c  geneSet" class="gt_row gt_left">GO:0032391</td>
<td headers="Bas-c  description" class="gt_row gt_left">photoreceptor connecting cilium</td>
<td headers="Bas-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Cl/G-a">Cl/G-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Cl/G-a  geneSet" class="gt_row gt_left">GO:0046578</td>
<td headers="Cl/G-a  description" class="gt_row gt_left">regulation of Ras protein signal transduction</td>
<td headers="Cl/G-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Cl/G-b">Cl/G-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Cl/G-b  geneSet" class="gt_row gt_left">GO:0006140</td>
<td headers="Cl/G-b  description" class="gt_row gt_left">regulation of nucleotide metabolic process</td>
<td headers="Cl/G-b  FDR" class="gt_row gt_left">0.05</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Hil">Hil</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Hil  geneSet" class="gt_row gt_left">GO:0042752</td>
<td headers="Hil  description" class="gt_row gt_left">regulation of circadian rhythm</td>
<td headers="Hil  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Hil  geneSet" class="gt_row gt_left">GO:0019730</td>
<td headers="Hil  description" class="gt_row gt_left">antimicrobial humoral response</td>
<td headers="Hil  FDR" class="gt_row gt_left">0.02</td></tr>
    <tr><td headers="Hil  geneSet" class="gt_row gt_left">GO:0045111</td>
<td headers="Hil  description" class="gt_row gt_left">intermediate filament cytoskeleton</td>
<td headers="Hil  FDR" class="gt_row gt_left">0.02</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Gob">Gob</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Gob  geneSet" class="gt_row gt_left">GO:0090407</td>
<td headers="Gob  description" class="gt_row gt_left">organophosphate biosynthetic process</td>
<td headers="Gob  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Gob1">Gob1</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Gob1  geneSet" class="gt_row gt_left">GO:0033674</td>
<td headers="Gob1  description" class="gt_row gt_left">positive regulation of kinase activity</td>
<td headers="Gob1  FDR" class="gt_row gt_left">0.05</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Gob2">Gob2</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Gob2  geneSet" class="gt_row gt_left">GO:0006996</td>
<td headers="Gob2  description" class="gt_row gt_left">organelle organization</td>
<td headers="Gob2  FDR" class="gt_row gt_left">0.14</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tuf1">Tuf1</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tuf1  geneSet" class="gt_row gt_left">GO:0098798</td>
<td headers="Tuf1  description" class="gt_row gt_left">mitochondrial protein complex</td>
<td headers="Tuf1  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Tuf1  geneSet" class="gt_row gt_left">GO:0098800</td>
<td headers="Tuf1  description" class="gt_row gt_left">inner mitochondrial membrane protein complex</td>
<td headers="Tuf1  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tuf2">Tuf2</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tuf2  geneSet" class="gt_row gt_left">GO:0005198</td>
<td headers="Tuf2  description" class="gt_row gt_left">structural molecule activity</td>
<td headers="Tuf2  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Nec">Nec</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Nec  geneSet" class="gt_row gt_left">GO:0005088</td>
<td headers="Nec  description" class="gt_row gt_left">Ras guanyl-nucleotide exchange factor activity</td>
<td headers="Nec  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Ion">Ion</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Ion  geneSet" class="gt_row gt_left">GO:0015078</td>
<td headers="Ion  description" class="gt_row gt_left">proton transmembrane transporter activity</td>
<td headers="Ion  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Ion  geneSet" class="gt_row gt_left">GO:0009055</td>
<td headers="Ion  description" class="gt_row gt_left">electron transfer activity</td>
<td headers="Ion  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Ion  geneSet" class="gt_row gt_left">GO:1902600</td>
<td headers="Ion  description" class="gt_row gt_left">proton transmembrane transport</td>
<td headers="Ion  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Ion  geneSet" class="gt_row gt_left">GO:0016469</td>
<td headers="Ion  description" class="gt_row gt_left">proton-transporting two-sector ATPase complex</td>
<td headers="Ion  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Ion  geneSet" class="gt_row gt_left">GO:0036442</td>
<td headers="Ion  description" class="gt_row gt_left">proton-exporting ATPase activity</td>
<td headers="Ion  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tp0">Tp0</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tp0  geneSet" class="gt_row gt_left">GO:0060021</td>
<td headers="Tp0  description" class="gt_row gt_left">roof of mouth development</td>
<td headers="Tp0  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Tp0  geneSet" class="gt_row gt_left">GO:0042274</td>
<td headers="Tp0  description" class="gt_row gt_left">ribosomal small subunit biogenesis</td>
<td headers="Tp0  FDR" class="gt_row gt_left">0.02</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tp30">Tp30</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tp30  geneSet" class="gt_row gt_left">GO:0008234</td>
<td headers="Tp30  description" class="gt_row gt_left">cysteine-type peptidase activity</td>
<td headers="Tp30  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Tp30  geneSet" class="gt_row gt_left">GO:0051606</td>
<td headers="Tp30  description" class="gt_row gt_left">detection of stimulus</td>
<td headers="Tp30  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tp60">Tp60</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tp60  geneSet" class="gt_row gt_left">GO:0043618</td>
<td headers="Tp60  description" class="gt_row gt_left">regulation of transcription from RNA polymerase II promoter in response to stress</td>
<td headers="Tp60  FDR" class="gt_row gt_left">0.03</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="R1xTp30">R1xTp30</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="R1xTp30  geneSet" class="gt_row gt_left">GO:0002821</td>
<td headers="R1xTp30  description" class="gt_row gt_left">positive regulation of adaptive immune response</td>
<td headers="R1xTp30  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="R1xTp30  geneSet" class="gt_row gt_left">GO:0005791</td>
<td headers="R1xTp30  description" class="gt_row gt_left">rough endoplasmic reticulum</td>
<td headers="R1xTp30  FDR" class="gt_row gt_left">0.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Dns-a">Dns-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Dns-a  geneSet" class="gt_row gt_left">GO:0001655</td>
<td headers="Dns-a  description" class="gt_row gt_left">urogenital system development</td>
<td headers="Dns-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Dns-a  geneSet" class="gt_row gt_left">GO:0001822</td>
<td headers="Dns-a  description" class="gt_row gt_left">kidney development</td>
<td headers="Dns-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Dns-a  geneSet" class="gt_row gt_left">GO:0007162</td>
<td headers="Dns-a  description" class="gt_row gt_left">negative regulation of cell adhesion</td>
<td headers="Dns-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Dns-a  geneSet" class="gt_row gt_left">GO:0022626</td>
<td headers="Dns-a  description" class="gt_row gt_left">cytosolic ribosome</td>
<td headers="Dns-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Dns-a  geneSet" class="gt_row gt_left">GO:0043604</td>
<td headers="Dns-a  description" class="gt_row gt_left">amide biosynthetic process</td>
<td headers="Dns-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Dns-a  geneSet" class="gt_row gt_left">GO:0072001</td>
<td headers="Dns-a  description" class="gt_row gt_left">renal system development</td>
<td headers="Dns-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Dns-a  geneSet" class="gt_row gt_left">GO:1990904</td>
<td headers="Dns-a  description" class="gt_row gt_left">ribonucleoprotein complex</td>
<td headers="Dns-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Dns-b">Dns-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Dns-b  geneSet" class="gt_row gt_left">GO:0031072</td>
<td headers="Dns-b  description" class="gt_row gt_left">heat shock protein binding</td>
<td headers="Dns-b  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Dns-b  geneSet" class="gt_row gt_left">GO:0034470</td>
<td headers="Dns-b  description" class="gt_row gt_left">ncRNA processing</td>
<td headers="Dns-b  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Dns-b  geneSet" class="gt_row gt_left">GO:0007018</td>
<td headers="Dns-b  description" class="gt_row gt_left">microtubule-based movement</td>
<td headers="Dns-b  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Dns-b  geneSet" class="gt_row gt_left">GO:0034605</td>
<td headers="Dns-b  description" class="gt_row gt_left">cellular response to heat</td>
<td headers="Dns-b  FDR" class="gt_row gt_left">0.02</td></tr>
    <tr><td headers="Dns-b  geneSet" class="gt_row gt_left">GO:0006606</td>
<td headers="Dns-b  description" class="gt_row gt_left">protein import into nucleus</td>
<td headers="Dns-b  FDR" class="gt_row gt_left">0.02</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Dns-c">Dns-c</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Dns-c  geneSet" class="gt_row gt_left">GO:0004364</td>
<td headers="Dns-c  description" class="gt_row gt_left">glutathione transferase activity</td>
<td headers="Dns-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Dns-c  geneSet" class="gt_row gt_left">GO:0060828</td>
<td headers="Dns-c  description" class="gt_row gt_left">regulation of canonical Wnt signaling pathway</td>
<td headers="Dns-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Dns-c  geneSet" class="gt_row gt_left">GO:0072341</td>
<td headers="Dns-c  description" class="gt_row gt_left">modified amino acid binding</td>
<td headers="Dns-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Dns-c  geneSet" class="gt_row gt_left">GO:0071466</td>
<td headers="Dns-c  description" class="gt_row gt_left">cellular response to xenobiotic stimulus</td>
<td headers="Dns-c  FDR" class="gt_row gt_left">0.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Dns-d">Dns-d</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Dns-d  geneSet" class="gt_row gt_left">GO:0008509</td>
<td headers="Dns-d  description" class="gt_row gt_left">anion transmembrane transporter activity</td>
<td headers="Dns-d  FDR" class="gt_row gt_left">0.21</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Sprs-a">Sprs-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Sprs-a  geneSet" class="gt_row gt_left">GO:0034660</td>
<td headers="Sprs-a  description" class="gt_row gt_left">ncRNA metabolic process</td>
<td headers="Sprs-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Sprs-a  geneSet" class="gt_row gt_left">GO:0016072</td>
<td headers="Sprs-a  description" class="gt_row gt_left">rRNA metabolic process</td>
<td headers="Sprs-a  FDR" class="gt_row gt_left">0.02</td></tr>
    <tr><td headers="Sprs-a  geneSet" class="gt_row gt_left">GO:0015934</td>
<td headers="Sprs-a  description" class="gt_row gt_left">large ribosomal subunit</td>
<td headers="Sprs-a  FDR" class="gt_row gt_left">0.04</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Sprs-b">Sprs-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Sprs-b  geneSet" class="gt_row gt_left">GO:0003008</td>
<td headers="Sprs-b  description" class="gt_row gt_left">system process</td>
<td headers="Sprs-b  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Sprs-c">Sprs-c</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Sprs-c  geneSet" class="gt_row gt_left">GO:0043087</td>
<td headers="Sprs-c  description" class="gt_row gt_left">regulation of GTPase activity</td>
<td headers="Sprs-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
  </tbody>
  
  
</table>
</div>
</div>
<div id="nmf-annotations" class="section level2">
<h2>NMF annotations</h2>
<pre class="r"><code>nmf_GSEA &lt;- readRDS(&quot;output/montoro-nmf-gsea.rds&quot;)
make_tbl(nmf_GSEA, nnlm_ksort[c(12:30, 5:11, 1:4)])</code></pre>
<div id="pvmnmoqvpp" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#pvmnmoqvpp table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#pvmnmoqvpp thead, #pvmnmoqvpp tbody, #pvmnmoqvpp tfoot, #pvmnmoqvpp tr, #pvmnmoqvpp td, #pvmnmoqvpp th {
  border-style: none;
}

#pvmnmoqvpp p {
  margin: 0;
  padding: 0;
}

#pvmnmoqvpp .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 11px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#pvmnmoqvpp .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#pvmnmoqvpp .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#pvmnmoqvpp .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#pvmnmoqvpp .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#pvmnmoqvpp .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#pvmnmoqvpp .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#pvmnmoqvpp .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#pvmnmoqvpp .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#pvmnmoqvpp .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#pvmnmoqvpp .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#pvmnmoqvpp .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#pvmnmoqvpp .gt_spanner_row {
  border-bottom-style: hidden;
}

#pvmnmoqvpp .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#pvmnmoqvpp .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#pvmnmoqvpp .gt_from_md > :first-child {
  margin-top: 0;
}

#pvmnmoqvpp .gt_from_md > :last-child {
  margin-bottom: 0;
}

#pvmnmoqvpp .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#pvmnmoqvpp .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#pvmnmoqvpp .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#pvmnmoqvpp .gt_row_group_first td {
  border-top-width: 2px;
}

#pvmnmoqvpp .gt_row_group_first th {
  border-top-width: 2px;
}

#pvmnmoqvpp .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#pvmnmoqvpp .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#pvmnmoqvpp .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#pvmnmoqvpp .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#pvmnmoqvpp .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#pvmnmoqvpp .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#pvmnmoqvpp .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#pvmnmoqvpp .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#pvmnmoqvpp .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#pvmnmoqvpp .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#pvmnmoqvpp .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#pvmnmoqvpp .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#pvmnmoqvpp .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#pvmnmoqvpp .gt_left {
  text-align: left;
}

#pvmnmoqvpp .gt_center {
  text-align: center;
}

#pvmnmoqvpp .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#pvmnmoqvpp .gt_font_normal {
  font-weight: normal;
}

#pvmnmoqvpp .gt_font_bold {
  font-weight: bold;
}

#pvmnmoqvpp .gt_font_italic {
  font-style: italic;
}

#pvmnmoqvpp .gt_super {
  font-size: 65%;
}

#pvmnmoqvpp .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#pvmnmoqvpp .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#pvmnmoqvpp .gt_indent_1 {
  text-indent: 5px;
}

#pvmnmoqvpp .gt_indent_2 {
  text-indent: 10px;
}

#pvmnmoqvpp .gt_indent_3 {
  text-indent: 15px;
}

#pvmnmoqvpp .gt_indent_4 {
  text-indent: 20px;
}

#pvmnmoqvpp .gt_indent_5 {
  text-indent: 25px;
}

#pvmnmoqvpp .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#pvmnmoqvpp div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>
<table class="gt_table" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
  <thead>
    <tr class="gt_col_headings">
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="geneSet">Gene Set</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="description">Description</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="FDR">FDR</th>
    </tr>
  </thead>
  <tbody class="gt_table_body">
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Cil-c, Cl/G-b">Cil-c, Cl/G-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Cil-c, Cl/G-b  geneSet" class="gt_row gt_left">GO:0021537</td>
<td headers="Cil-c, Cl/G-b  description" class="gt_row gt_left">telencephalon development</td>
<td headers="Cil-c, Cl/G-b  FDR" class="gt_row gt_left">&lt;.01, &lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Prl-b, Bas-a, Bas-b, Tp0-a, Dns-a, Dns-d">Prl-b, Bas-a, Bas-b, Tp0-a, Dns-a, Dns-d</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Prl-b, Bas-a, Bas-b, Tp0-a, Dns-a, Dns-d  geneSet" class="gt_row gt_left">GO:0005840</td>
<td headers="Prl-b, Bas-a, Bas-b, Tp0-a, Dns-a, Dns-d  description" class="gt_row gt_left">ribosome</td>
<td headers="Prl-b, Bas-a, Bas-b, Tp0-a, Dns-a, Dns-d  FDR" class="gt_row gt_left">all &lt; 0.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Prl-b, Bas-a, Tp0-a">Prl-b, Bas-a, Tp0-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Prl-b, Bas-a, Tp0-a  geneSet" class="gt_row gt_left">GO:0022627</td>
<td headers="Prl-b, Bas-a, Tp0-a  description" class="gt_row gt_left">cytosolic small ribosomal subunit</td>
<td headers="Prl-b, Bas-a, Tp0-a  FDR" class="gt_row gt_left">&lt;.01, &lt;.01, &lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Prl-b, Bas-a, Tuf, Tp0-a">Prl-b, Bas-a, Tuf, Tp0-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Prl-b, Bas-a, Tuf, Tp0-a  geneSet" class="gt_row gt_left">GO:0044445</td>
<td headers="Prl-b, Bas-a, Tuf, Tp0-a  description" class="gt_row gt_left">cytosolic part</td>
<td headers="Prl-b, Bas-a, Tuf, Tp0-a  FDR" class="gt_row gt_left">all &lt; 0.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Prl-b, Cl/G-b">Prl-b, Cl/G-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Prl-b, Cl/G-b  geneSet" class="gt_row gt_left">GO:0015935</td>
<td headers="Prl-b, Cl/G-b  description" class="gt_row gt_left">small ribosomal subunit</td>
<td headers="Prl-b, Cl/G-b  FDR" class="gt_row gt_left">&lt;.01, &lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Prl-b, Tp0-a, Dns-a">Prl-b, Tp0-a, Dns-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Prl-b, Tp0-a, Dns-a  geneSet" class="gt_row gt_left">GO:0044391</td>
<td headers="Prl-b, Tp0-a, Dns-a  description" class="gt_row gt_left">ribosomal subunit</td>
<td headers="Prl-b, Tp0-a, Dns-a  FDR" class="gt_row gt_left">&lt;.01, &lt;.01, &lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Prl-c, Dns-c">Prl-c, Dns-c</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Prl-c, Dns-c  geneSet" class="gt_row gt_left">GO:0006364</td>
<td headers="Prl-c, Dns-c  description" class="gt_row gt_left">rRNA processing</td>
<td headers="Prl-c, Dns-c  FDR" class="gt_row gt_left">0.01, 0.03</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Bas-a, Tuf, Dns-d">Bas-a, Tuf, Dns-d</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Bas-a, Tuf, Dns-d  geneSet" class="gt_row gt_left">GO:0003735</td>
<td headers="Bas-a, Tuf, Dns-d  description" class="gt_row gt_left">structural constituent of ribosome</td>
<td headers="Bas-a, Tuf, Dns-d  FDR" class="gt_row gt_left">&lt;.01, &lt;.01, &lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Bas-b, Dns-a">Bas-b, Dns-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Bas-b, Dns-a  geneSet" class="gt_row gt_left">GO:1990904</td>
<td headers="Bas-b, Dns-a  description" class="gt_row gt_left">ribonucleoprotein complex</td>
<td headers="Bas-b, Dns-a  FDR" class="gt_row gt_left">&lt;.01, &lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Bas-b, Dns-d">Bas-b, Dns-d</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Bas-b, Dns-d  geneSet" class="gt_row gt_left">GO:0005198</td>
<td headers="Bas-b, Dns-d  description" class="gt_row gt_left">structural molecule activity</td>
<td headers="Bas-b, Dns-d  FDR" class="gt_row gt_left">&lt;.01, &lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Bas-c, Tp0-b">Bas-c, Tp0-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Bas-c, Tp0-b  geneSet" class="gt_row gt_left">GO:0035869</td>
<td headers="Bas-c, Tp0-b  description" class="gt_row gt_left">ciliary transition zone</td>
<td headers="Bas-c, Tp0-b  FDR" class="gt_row gt_left">0.01, 0.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Bas-d, Cl/G-b">Bas-d, Cl/G-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Bas-d, Cl/G-b  geneSet" class="gt_row gt_left">GO:0003714</td>
<td headers="Bas-d, Cl/G-b  description" class="gt_row gt_left">transcription corepressor activity</td>
<td headers="Bas-d, Cl/G-b  FDR" class="gt_row gt_left">0.08, &lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Bas-e, Tuf">Bas-e, Tuf</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Bas-e, Tuf  geneSet" class="gt_row gt_left">GO:0098800</td>
<td headers="Bas-e, Tuf  description" class="gt_row gt_left">inner mitochondrial membrane protein complex</td>
<td headers="Bas-e, Tuf  FDR" class="gt_row gt_left">0.05, &lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tp0-a, Dns-a">Tp0-a, Dns-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tp0-a, Dns-a  geneSet" class="gt_row gt_left">GO:0022626</td>
<td headers="Tp0-a, Dns-a  description" class="gt_row gt_left">cytosolic ribosome</td>
<td headers="Tp0-a, Dns-a  FDR" class="gt_row gt_left">&lt;.01, &lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Cil-a">Cil-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Cil-a  geneSet" class="gt_row gt_left">GO:0003300</td>
<td headers="Cil-a  description" class="gt_row gt_left">cardiac muscle hypertrophy</td>
<td headers="Cil-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Cil-b">Cil-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Cil-b  geneSet" class="gt_row gt_left">GO:0042274</td>
<td headers="Cil-b  description" class="gt_row gt_left">ribosomal small subunit biogenesis</td>
<td headers="Cil-b  FDR" class="gt_row gt_left">0.06</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Cil-c">Cil-c</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Cil-c  geneSet" class="gt_row gt_left">GO:0099536</td>
<td headers="Cil-c  description" class="gt_row gt_left">synaptic signaling</td>
<td headers="Cil-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Cil-c  geneSet" class="gt_row gt_left">GO:0050804</td>
<td headers="Cil-c  description" class="gt_row gt_left">modulation of chemical synaptic transmission</td>
<td headers="Cil-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Prl-a">Prl-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Prl-a  geneSet" class="gt_row gt_left">GO:0004888</td>
<td headers="Prl-a  description" class="gt_row gt_left">transmembrane signaling receptor activity</td>
<td headers="Prl-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Prl-a  geneSet" class="gt_row gt_left">GO:0030992</td>
<td headers="Prl-a  description" class="gt_row gt_left">intraciliary transport particle B</td>
<td headers="Prl-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Prl-a  geneSet" class="gt_row gt_left">GO:0007224</td>
<td headers="Prl-a  description" class="gt_row gt_left">smoothened signaling pathway</td>
<td headers="Prl-a  FDR" class="gt_row gt_left">0.02</td></tr>
    <tr><td headers="Prl-a  geneSet" class="gt_row gt_left">GO:0048771</td>
<td headers="Prl-a  description" class="gt_row gt_left">tissue remodeling</td>
<td headers="Prl-a  FDR" class="gt_row gt_left">0.05</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Bas-a">Bas-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Bas-a  geneSet" class="gt_row gt_left">GO:0015934</td>
<td headers="Bas-a  description" class="gt_row gt_left">large ribosomal subunit</td>
<td headers="Bas-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Bas-a  geneSet" class="gt_row gt_left">GO:0019843</td>
<td headers="Bas-a  description" class="gt_row gt_left">rRNA binding</td>
<td headers="Bas-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Bas-a  geneSet" class="gt_row gt_left">GO:0022625</td>
<td headers="Bas-a  description" class="gt_row gt_left">cytosolic large ribosomal subunit</td>
<td headers="Bas-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Bas-b">Bas-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Bas-b  geneSet" class="gt_row gt_left">GO:0030141</td>
<td headers="Bas-b  description" class="gt_row gt_left">secretory granule</td>
<td headers="Bas-b  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Bas-b  geneSet" class="gt_row gt_left">GO:0035914</td>
<td headers="Bas-b  description" class="gt_row gt_left">skeletal muscle cell differentiation</td>
<td headers="Bas-b  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Bas-c">Bas-c</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Bas-c  geneSet" class="gt_row gt_left">GO:0015267</td>
<td headers="Bas-c  description" class="gt_row gt_left">channel activity</td>
<td headers="Bas-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Bas-c  geneSet" class="gt_row gt_left">GO:0042073</td>
<td headers="Bas-c  description" class="gt_row gt_left">intraciliary transport</td>
<td headers="Bas-c  FDR" class="gt_row gt_left">0.02</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Bas-e">Bas-e</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Bas-e  geneSet" class="gt_row gt_left">GO:0001508</td>
<td headers="Bas-e  description" class="gt_row gt_left">action potential</td>
<td headers="Bas-e  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Cl/G-a">Cl/G-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Cl/G-a  geneSet" class="gt_row gt_left">GO:0006457</td>
<td headers="Cl/G-a  description" class="gt_row gt_left">protein folding</td>
<td headers="Cl/G-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Cl/G-a  geneSet" class="gt_row gt_left">GO:0042176</td>
<td headers="Cl/G-a  description" class="gt_row gt_left">regulation of protein catabolic process</td>
<td headers="Cl/G-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Cl/G-a  geneSet" class="gt_row gt_left">GO:0051082</td>
<td headers="Cl/G-a  description" class="gt_row gt_left">unfolded protein binding</td>
<td headers="Cl/G-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Cl/G-a  geneSet" class="gt_row gt_left">GO:0061077</td>
<td headers="Cl/G-a  description" class="gt_row gt_left">chaperone-mediated protein folding</td>
<td headers="Cl/G-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Cl/G-a  geneSet" class="gt_row gt_left">GO:0030163</td>
<td headers="Cl/G-a  description" class="gt_row gt_left">protein catabolic process</td>
<td headers="Cl/G-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Cl/G-b">Cl/G-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Cl/G-b  geneSet" class="gt_row gt_left">GO:0045727</td>
<td headers="Cl/G-b  description" class="gt_row gt_left">positive regulation of translation</td>
<td headers="Cl/G-b  FDR" class="gt_row gt_left">0.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Cl/G-c">Cl/G-c</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Cl/G-c  geneSet" class="gt_row gt_left">GO:0001558</td>
<td headers="Cl/G-c  description" class="gt_row gt_left">regulation of cell growth</td>
<td headers="Cl/G-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Cl/G-c  geneSet" class="gt_row gt_left">GO:0010594</td>
<td headers="Cl/G-c  description" class="gt_row gt_left">regulation of endothelial cell migration</td>
<td headers="Cl/G-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Cl/G-c  geneSet" class="gt_row gt_left">GO:0090407</td>
<td headers="Cl/G-c  description" class="gt_row gt_left">organophosphate biosynthetic process</td>
<td headers="Cl/G-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Cl/G-c  geneSet" class="gt_row gt_left">GO:0030307</td>
<td headers="Cl/G-c  description" class="gt_row gt_left">positive regulation of cell growth</td>
<td headers="Cl/G-c  FDR" class="gt_row gt_left">0.02</td></tr>
    <tr><td headers="Cl/G-c  geneSet" class="gt_row gt_left">GO:0016049</td>
<td headers="Cl/G-c  description" class="gt_row gt_left">cell growth</td>
<td headers="Cl/G-c  FDR" class="gt_row gt_left">0.03</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Hil-a">Hil-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Hil-a  geneSet" class="gt_row gt_left">GO:0005791</td>
<td headers="Hil-a  description" class="gt_row gt_left">rough endoplasmic reticulum</td>
<td headers="Hil-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Hil-a  geneSet" class="gt_row gt_left">GO:0001818</td>
<td headers="Hil-a  description" class="gt_row gt_left">negative regulation of cytokine production</td>
<td headers="Hil-a  FDR" class="gt_row gt_left">0.04</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Hil-b">Hil-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Hil-b  geneSet" class="gt_row gt_left">GO:0006164</td>
<td headers="Hil-b  description" class="gt_row gt_left">purine nucleotide biosynthetic process</td>
<td headers="Hil-b  FDR" class="gt_row gt_left">0.03</td></tr>
    <tr><td headers="Hil-b  geneSet" class="gt_row gt_left">GO:0098798</td>
<td headers="Hil-b  description" class="gt_row gt_left">mitochondrial protein complex</td>
<td headers="Hil-b  FDR" class="gt_row gt_left">0.04</td></tr>
    <tr><td headers="Hil-b  geneSet" class="gt_row gt_left">GO:0044455</td>
<td headers="Hil-b  description" class="gt_row gt_left">mitochondrial membrane part</td>
<td headers="Hil-b  FDR" class="gt_row gt_left">0.04</td></tr>
    <tr><td headers="Hil-b  geneSet" class="gt_row gt_left">GO:0005743</td>
<td headers="Hil-b  description" class="gt_row gt_left">mitochondrial inner membrane</td>
<td headers="Hil-b  FDR" class="gt_row gt_left">0.04</td></tr>
    <tr><td headers="Hil-b  geneSet" class="gt_row gt_left">GO:0006403</td>
<td headers="Hil-b  description" class="gt_row gt_left">RNA localization</td>
<td headers="Hil-b  FDR" class="gt_row gt_left">0.05</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tuf">Tuf</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tuf  geneSet" class="gt_row gt_left">GO:0042089</td>
<td headers="Tuf  description" class="gt_row gt_left">cytokine biosynthetic process</td>
<td headers="Tuf  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Tuf  geneSet" class="gt_row gt_left">GO:0035249</td>
<td headers="Tuf  description" class="gt_row gt_left">synaptic transmission, glutamatergic</td>
<td headers="Tuf  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Nec">Nec</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Nec  geneSet" class="gt_row gt_left">GO:0005088</td>
<td headers="Nec  description" class="gt_row gt_left">Ras guanyl-nucleotide exchange factor activity</td>
<td headers="Nec  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Nec  geneSet" class="gt_row gt_left">GO:1901605</td>
<td headers="Nec  description" class="gt_row gt_left">alpha-amino acid metabolic process</td>
<td headers="Nec  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Ion/Cl/G">Ion/Cl/G</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Ion/Cl/G  geneSet" class="gt_row gt_left">GO:0051223</td>
<td headers="Ion/Cl/G  description" class="gt_row gt_left">regulation of protein transport</td>
<td headers="Ion/Cl/G  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Ion/Cl/G  geneSet" class="gt_row gt_left">GO:0006511</td>
<td headers="Ion/Cl/G  description" class="gt_row gt_left">ubiquitin-dependent protein catabolic process</td>
<td headers="Ion/Cl/G  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Ion/Cl/G  geneSet" class="gt_row gt_left">GO:0051271</td>
<td headers="Ion/Cl/G  description" class="gt_row gt_left">negative regulation of cellular component movement</td>
<td headers="Ion/Cl/G  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Ion/Cl/G  geneSet" class="gt_row gt_left">GO:0015833</td>
<td headers="Ion/Cl/G  description" class="gt_row gt_left">peptide transport</td>
<td headers="Ion/Cl/G  FDR" class="gt_row gt_left">0.02</td></tr>
    <tr><td headers="Ion/Cl/G  geneSet" class="gt_row gt_left">GO:0016323</td>
<td headers="Ion/Cl/G  description" class="gt_row gt_left">basolateral plasma membrane</td>
<td headers="Ion/Cl/G  FDR" class="gt_row gt_left">0.03</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="TpxCl">TpxCl</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="TpxCl  geneSet" class="gt_row gt_left">GO:0043618</td>
<td headers="TpxCl  description" class="gt_row gt_left">regulation of transcription from RNA polymerase II promoter in response to stress</td>
<td headers="TpxCl  FDR" class="gt_row gt_left">0.13</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tp30/60">Tp30/60</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tp30/60  geneSet" class="gt_row gt_left">GO:0001894</td>
<td headers="Tp30/60  description" class="gt_row gt_left">tissue homeostasis</td>
<td headers="Tp30/60  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tp0-a">Tp0-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tp0-a  geneSet" class="gt_row gt_left">GO:0043583</td>
<td headers="Tp0-a  description" class="gt_row gt_left">ear development</td>
<td headers="Tp0-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tp0-b">Tp0-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tp0-b  geneSet" class="gt_row gt_left">GO:0030990</td>
<td headers="Tp0-b  description" class="gt_row gt_left">intraciliary transport particle</td>
<td headers="Tp0-b  FDR" class="gt_row gt_left">0.02</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tp30">Tp30</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tp30  geneSet" class="gt_row gt_left">GO:0046777</td>
<td headers="Tp30  description" class="gt_row gt_left">protein autophosphorylation</td>
<td headers="Tp30  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tp60">Tp60</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tp60  geneSet" class="gt_row gt_left">GO:0050803</td>
<td headers="Tp60  description" class="gt_row gt_left">regulation of synapse structure or activity</td>
<td headers="Tp60  FDR" class="gt_row gt_left">0.02</td></tr>
    <tr><td headers="Tp60  geneSet" class="gt_row gt_left">GO:0042737</td>
<td headers="Tp60  description" class="gt_row gt_left">drug catabolic process</td>
<td headers="Tp60  FDR" class="gt_row gt_left">0.03</td></tr>
    <tr><td headers="Tp60  geneSet" class="gt_row gt_left">GO:0099173</td>
<td headers="Tp60  description" class="gt_row gt_left">postsynapse organization</td>
<td headers="Tp60  FDR" class="gt_row gt_left">0.04</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="R1xTp30">R1xTp30</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="R1xTp30  geneSet" class="gt_row gt_left">GO:0002263</td>
<td headers="R1xTp30  description" class="gt_row gt_left">cell activation involved in immune response</td>
<td headers="R1xTp30  FDR" class="gt_row gt_left">0.59</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Dns-a">Dns-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Dns-a  geneSet" class="gt_row gt_left">GO:0006412</td>
<td headers="Dns-a  description" class="gt_row gt_left">translation</td>
<td headers="Dns-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Dns-b">Dns-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Dns-b  geneSet" class="gt_row gt_left">GO:0009986</td>
<td headers="Dns-b  description" class="gt_row gt_left">cell surface</td>
<td headers="Dns-b  FDR" class="gt_row gt_left">0.06</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Dns-c">Dns-c</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Dns-c  geneSet" class="gt_row gt_left">GO:0034660</td>
<td headers="Dns-c  description" class="gt_row gt_left">ncRNA metabolic process</td>
<td headers="Dns-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Dns-c  geneSet" class="gt_row gt_left">GO:0048037</td>
<td headers="Dns-c  description" class="gt_row gt_left">cofactor binding</td>
<td headers="Dns-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Dns-c  geneSet" class="gt_row gt_left">GO:0030111</td>
<td headers="Dns-c  description" class="gt_row gt_left">regulation of Wnt signaling pathway</td>
<td headers="Dns-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Dns-c  geneSet" class="gt_row gt_left">GO:0034470</td>
<td headers="Dns-c  description" class="gt_row gt_left">ncRNA processing</td>
<td headers="Dns-c  FDR" class="gt_row gt_left">0.04</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Dns-d">Dns-d</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Dns-d  geneSet" class="gt_row gt_left">GO:0003015</td>
<td headers="Dns-d  description" class="gt_row gt_left">heart process</td>
<td headers="Dns-d  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Dns-d  geneSet" class="gt_row gt_left">GO:0010817</td>
<td headers="Dns-d  description" class="gt_row gt_left">regulation of hormone levels</td>
<td headers="Dns-d  FDR" class="gt_row gt_left">&lt;.01</td></tr>
  </tbody>
  
  
</table>
</div>
</div>
<div id="topic-model-annotations" class="section level2">
<h2>Topic model annotations</h2>
<pre class="r"><code>tm_GSEA &lt;- readRDS(&quot;output/montoro-tm-gsea.rds&quot;)
make_tbl(tm_GSEA, tm_ksort[c(16:30, 6:15, 1:5)])</code></pre>
<div id="nrfjqrpdmr" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#nrfjqrpdmr table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#nrfjqrpdmr thead, #nrfjqrpdmr tbody, #nrfjqrpdmr tfoot, #nrfjqrpdmr tr, #nrfjqrpdmr td, #nrfjqrpdmr th {
  border-style: none;
}

#nrfjqrpdmr p {
  margin: 0;
  padding: 0;
}

#nrfjqrpdmr .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 11px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#nrfjqrpdmr .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#nrfjqrpdmr .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#nrfjqrpdmr .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#nrfjqrpdmr .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#nrfjqrpdmr .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#nrfjqrpdmr .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#nrfjqrpdmr .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#nrfjqrpdmr .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#nrfjqrpdmr .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#nrfjqrpdmr .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#nrfjqrpdmr .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#nrfjqrpdmr .gt_spanner_row {
  border-bottom-style: hidden;
}

#nrfjqrpdmr .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#nrfjqrpdmr .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#nrfjqrpdmr .gt_from_md > :first-child {
  margin-top: 0;
}

#nrfjqrpdmr .gt_from_md > :last-child {
  margin-bottom: 0;
}

#nrfjqrpdmr .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#nrfjqrpdmr .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#nrfjqrpdmr .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#nrfjqrpdmr .gt_row_group_first td {
  border-top-width: 2px;
}

#nrfjqrpdmr .gt_row_group_first th {
  border-top-width: 2px;
}

#nrfjqrpdmr .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#nrfjqrpdmr .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#nrfjqrpdmr .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#nrfjqrpdmr .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#nrfjqrpdmr .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#nrfjqrpdmr .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#nrfjqrpdmr .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#nrfjqrpdmr .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#nrfjqrpdmr .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#nrfjqrpdmr .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#nrfjqrpdmr .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#nrfjqrpdmr .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#nrfjqrpdmr .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#nrfjqrpdmr .gt_left {
  text-align: left;
}

#nrfjqrpdmr .gt_center {
  text-align: center;
}

#nrfjqrpdmr .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#nrfjqrpdmr .gt_font_normal {
  font-weight: normal;
}

#nrfjqrpdmr .gt_font_bold {
  font-weight: bold;
}

#nrfjqrpdmr .gt_font_italic {
  font-style: italic;
}

#nrfjqrpdmr .gt_super {
  font-size: 65%;
}

#nrfjqrpdmr .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#nrfjqrpdmr .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#nrfjqrpdmr .gt_indent_1 {
  text-indent: 5px;
}

#nrfjqrpdmr .gt_indent_2 {
  text-indent: 10px;
}

#nrfjqrpdmr .gt_indent_3 {
  text-indent: 15px;
}

#nrfjqrpdmr .gt_indent_4 {
  text-indent: 20px;
}

#nrfjqrpdmr .gt_indent_5 {
  text-indent: 25px;
}

#nrfjqrpdmr .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#nrfjqrpdmr div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>
<table class="gt_table" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
  <thead>
    <tr class="gt_col_headings">
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="geneSet">Gene Set</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="description">Description</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="FDR">FDR</th>
    </tr>
  </thead>
  <tbody class="gt_table_body">
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Prl, B/P-b">Prl, B/P-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Prl, B/P-b  geneSet" class="gt_row gt_left">GO:0008203</td>
<td headers="Prl, B/P-b  description" class="gt_row gt_left">cholesterol metabolic process</td>
<td headers="Prl, B/P-b  FDR" class="gt_row gt_left">0.05, &lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Cl/Hil, Gob2">Cl/Hil, Gob2</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Cl/Hil, Gob2  geneSet" class="gt_row gt_left">GO:0005840</td>
<td headers="Cl/Hil, Gob2  description" class="gt_row gt_left">ribosome</td>
<td headers="Cl/Hil, Gob2  FDR" class="gt_row gt_left">&lt;.01, &lt;.01</td></tr>
    <tr><td headers="Cl/Hil, Gob2  geneSet" class="gt_row gt_left">GO:0022626</td>
<td headers="Cl/Hil, Gob2  description" class="gt_row gt_left">cytosolic ribosome</td>
<td headers="Cl/Hil, Gob2  FDR" class="gt_row gt_left">&lt;.01, &lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Gob1, Gob2">Gob1, Gob2</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Gob1, Gob2  geneSet" class="gt_row gt_left">GO:0044445</td>
<td headers="Gob1, Gob2  description" class="gt_row gt_left">cytosolic part</td>
<td headers="Gob1, Gob2  FDR" class="gt_row gt_left">&lt;.01, &lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Cil">Cil</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Cil  geneSet" class="gt_row gt_left">GO:0030534</td>
<td headers="Cil  description" class="gt_row gt_left">adult behavior</td>
<td headers="Cil  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Prl">Prl</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Prl  geneSet" class="gt_row gt_left">GO:0004888</td>
<td headers="Prl  description" class="gt_row gt_left">transmembrane signaling receptor activity</td>
<td headers="Prl  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Prl  geneSet" class="gt_row gt_left">GO:0007368</td>
<td headers="Prl  description" class="gt_row gt_left">determination of left/right symmetry</td>
<td headers="Prl  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Prl  geneSet" class="gt_row gt_left">GO:0005901</td>
<td headers="Prl  description" class="gt_row gt_left">caveola</td>
<td headers="Prl  FDR" class="gt_row gt_left">0.05</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="B/P-a">B/P-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="B/P-a  geneSet" class="gt_row gt_left">GO:0071229</td>
<td headers="B/P-a  description" class="gt_row gt_left">cellular response to acid chemical</td>
<td headers="B/P-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="B/P-b">B/P-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="B/P-b  geneSet" class="gt_row gt_left">GO:0016125</td>
<td headers="B/P-b  description" class="gt_row gt_left">sterol metabolic process</td>
<td headers="B/P-b  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="B/P-b  geneSet" class="gt_row gt_left">GO:0030183</td>
<td headers="B/P-b  description" class="gt_row gt_left">B cell differentiation</td>
<td headers="B/P-b  FDR" class="gt_row gt_left">0.02</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Bas">Bas</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Bas  geneSet" class="gt_row gt_left">GO:0035869</td>
<td headers="Bas  description" class="gt_row gt_left">ciliary transition zone</td>
<td headers="Bas  FDR" class="gt_row gt_left">0.03</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Cl/G">Cl/G</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Cl/G  geneSet" class="gt_row gt_left">GO:0042802</td>
<td headers="Cl/G  description" class="gt_row gt_left">identical protein binding</td>
<td headers="Cl/G  FDR" class="gt_row gt_left">0.25</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Club">Club</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Club  geneSet" class="gt_row gt_left">GO:0005743</td>
<td headers="Club  description" class="gt_row gt_left">mitochondrial inner membrane</td>
<td headers="Club  FDR" class="gt_row gt_left">0.27</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="H/G-a">H/G-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="H/G-a  geneSet" class="gt_row gt_left">GO:0007154</td>
<td headers="H/G-a  description" class="gt_row gt_left">cell communication</td>
<td headers="H/G-a  FDR" class="gt_row gt_left">0.33</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="H/G-b">H/G-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="H/G-b  geneSet" class="gt_row gt_left">GO:0046483</td>
<td headers="H/G-b  description" class="gt_row gt_left">heterocycle metabolic process</td>
<td headers="H/G-b  FDR" class="gt_row gt_left">0.49</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Hil">Hil</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Hil  geneSet" class="gt_row gt_left">GO:0008654</td>
<td headers="Hil  description" class="gt_row gt_left">phospholipid biosynthetic process</td>
<td headers="Hil  FDR" class="gt_row gt_left">0.11</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Gob1">Gob1</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Gob1  geneSet" class="gt_row gt_left">GO:0043228</td>
<td headers="Gob1  description" class="gt_row gt_left">non-membrane-bounded organelle</td>
<td headers="Gob1  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Gob1  geneSet" class="gt_row gt_left">GO:0043604</td>
<td headers="Gob1  description" class="gt_row gt_left">amide biosynthetic process</td>
<td headers="Gob1  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Gob1  geneSet" class="gt_row gt_left">GO:1990904</td>
<td headers="Gob1  description" class="gt_row gt_left">ribonucleoprotein complex</td>
<td headers="Gob1  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Gob2">Gob2</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Gob2  geneSet" class="gt_row gt_left">GO:0006396</td>
<td headers="Gob2  description" class="gt_row gt_left">RNA processing</td>
<td headers="Gob2  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Gob2  geneSet" class="gt_row gt_left">GO:0015935</td>
<td headers="Gob2  description" class="gt_row gt_left">small ribosomal subunit</td>
<td headers="Gob2  FDR" class="gt_row gt_left">0.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="TNI-a">TNI-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="TNI-a  geneSet" class="gt_row gt_left">GO:0007259</td>
<td headers="TNI-a  description" class="gt_row gt_left">JAK-STAT cascade</td>
<td headers="TNI-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="TNI-a  geneSet" class="gt_row gt_left">GO:0019226</td>
<td headers="TNI-a  description" class="gt_row gt_left">transmission of nerve impulse</td>
<td headers="TNI-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="TNI-a  geneSet" class="gt_row gt_left">GO:0008138</td>
<td headers="TNI-a  description" class="gt_row gt_left">protein tyrosine/serine/threonine phosphatase activity</td>
<td headers="TNI-a  FDR" class="gt_row gt_left">0.01</td></tr>
    <tr><td headers="TNI-a  geneSet" class="gt_row gt_left">GO:0071526</td>
<td headers="TNI-a  description" class="gt_row gt_left">semaphorin-plexin signaling pathway</td>
<td headers="TNI-a  FDR" class="gt_row gt_left">0.02</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="TNI-b">TNI-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="TNI-b  geneSet" class="gt_row gt_left">GO:0097305</td>
<td headers="TNI-b  description" class="gt_row gt_left">response to alcohol</td>
<td headers="TNI-b  FDR" class="gt_row gt_left">0.21</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="TpxCl">TpxCl</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="TpxCl  geneSet" class="gt_row gt_left">GO:0002526</td>
<td headers="TpxCl  description" class="gt_row gt_left">acute inflammatory response</td>
<td headers="TpxCl  FDR" class="gt_row gt_left">0.07</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tp0/60-a">Tp0/60-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tp0/60-a  geneSet" class="gt_row gt_left">GO:0007369</td>
<td headers="Tp0/60-a  description" class="gt_row gt_left">gastrulation</td>
<td headers="Tp0/60-a  FDR" class="gt_row gt_left">0.18</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tp0/60-b">Tp0/60-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tp0/60-b  geneSet" class="gt_row gt_left">GO:0001508</td>
<td headers="Tp0/60-b  description" class="gt_row gt_left">action potential</td>
<td headers="Tp0/60-b  FDR" class="gt_row gt_left">0.44</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tp30/60">Tp30/60</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tp30/60  geneSet" class="gt_row gt_left">GO:0014902</td>
<td headers="Tp30/60  description" class="gt_row gt_left">myotube differentiation</td>
<td headers="Tp30/60  FDR" class="gt_row gt_left">0.02</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tp0-a">Tp0-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tp0-a  geneSet" class="gt_row gt_left">GO:0007224</td>
<td headers="Tp0-a  description" class="gt_row gt_left">smoothened signaling pathway</td>
<td headers="Tp0-a  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tp0-b">Tp0-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tp0-b  geneSet" class="gt_row gt_left">GO:0001678</td>
<td headers="Tp0-b  description" class="gt_row gt_left">cellular glucose homeostasis</td>
<td headers="Tp0-b  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Tp0-b  geneSet" class="gt_row gt_left">GO:0007264</td>
<td headers="Tp0-b  description" class="gt_row gt_left">small GTPase mediated signal transduction</td>
<td headers="Tp0-b  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Tp0-b  geneSet" class="gt_row gt_left">GO:0033002</td>
<td headers="Tp0-b  description" class="gt_row gt_left">muscle cell proliferation</td>
<td headers="Tp0-b  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Tp0-b  geneSet" class="gt_row gt_left">GO:0071322</td>
<td headers="Tp0-b  description" class="gt_row gt_left">cellular response to carbohydrate stimulus</td>
<td headers="Tp0-b  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Tp0-b  geneSet" class="gt_row gt_left">GO:0071326</td>
<td headers="Tp0-b  description" class="gt_row gt_left">cellular response to monosaccharide stimulus</td>
<td headers="Tp0-b  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tp30xCl">Tp30xCl</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tp30xCl  geneSet" class="gt_row gt_left">GO:0046578</td>
<td headers="Tp30xCl  description" class="gt_row gt_left">regulation of Ras protein signal transduction</td>
<td headers="Tp30xCl  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Tp30xCl  geneSet" class="gt_row gt_left">GO:0032609</td>
<td headers="Tp30xCl  description" class="gt_row gt_left">interferon-gamma production</td>
<td headers="Tp30xCl  FDR" class="gt_row gt_left">0.02</td></tr>
    <tr><td headers="Tp30xCl  geneSet" class="gt_row gt_left">GO:0045580</td>
<td headers="Tp30xCl  description" class="gt_row gt_left">regulation of T cell differentiation</td>
<td headers="Tp30xCl  FDR" class="gt_row gt_left">0.04</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tp30-a">Tp30-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tp30-a  geneSet" class="gt_row gt_left">GO:0072330</td>
<td headers="Tp30-a  description" class="gt_row gt_left">monocarboxylic acid biosynthetic process</td>
<td headers="Tp30-a  FDR" class="gt_row gt_left">0.03</td></tr>
    <tr><td headers="Tp30-a  geneSet" class="gt_row gt_left">GO:1903076</td>
<td headers="Tp30-a  description" class="gt_row gt_left">regulation of protein localization to plasma membrane</td>
<td headers="Tp30-a  FDR" class="gt_row gt_left">0.03</td></tr>
    <tr><td headers="Tp30-a  geneSet" class="gt_row gt_left">GO:1904375</td>
<td headers="Tp30-a  description" class="gt_row gt_left">regulation of protein localization to cell periphery</td>
<td headers="Tp30-a  FDR" class="gt_row gt_left">0.04</td></tr>
    <tr><td headers="Tp30-a  geneSet" class="gt_row gt_left">GO:0019003</td>
<td headers="Tp30-a  description" class="gt_row gt_left">GDP binding</td>
<td headers="Tp30-a  FDR" class="gt_row gt_left">0.05</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tp30-b">Tp30-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tp30-b  geneSet" class="gt_row gt_left">GO:0004672</td>
<td headers="Tp30-b  description" class="gt_row gt_left">protein kinase activity</td>
<td headers="Tp30-b  FDR" class="gt_row gt_left">0.14</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Tp60">Tp60</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Tp60  geneSet" class="gt_row gt_left">GO:0016614</td>
<td headers="Tp60  description" class="gt_row gt_left">oxidoreductase activity, acting on CH-OH group of donors</td>
<td headers="Tp60  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Tp60  geneSet" class="gt_row gt_left">GO:0002028</td>
<td headers="Tp60  description" class="gt_row gt_left">regulation of sodium ion transport</td>
<td headers="Tp60  FDR" class="gt_row gt_left">0.02</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Dns-a">Dns-a</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Dns-a  geneSet" class="gt_row gt_left">GO:0022900</td>
<td headers="Dns-a  description" class="gt_row gt_left">electron transport chain</td>
<td headers="Dns-a  FDR" class="gt_row gt_left">0.05</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Dns-b">Dns-b</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Dns-b  geneSet" class="gt_row gt_left">GO:0001664</td>
<td headers="Dns-b  description" class="gt_row gt_left">G protein-coupled receptor binding</td>
<td headers="Dns-b  FDR" class="gt_row gt_left">0.22</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Dns-c">Dns-c</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Dns-c  geneSet" class="gt_row gt_left">GO:0001763</td>
<td headers="Dns-c  description" class="gt_row gt_left">morphogenesis of a branching structure</td>
<td headers="Dns-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Dns-c  geneSet" class="gt_row gt_left">GO:0061138</td>
<td headers="Dns-c  description" class="gt_row gt_left">morphogenesis of a branching epithelium</td>
<td headers="Dns-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr><td headers="Dns-c  geneSet" class="gt_row gt_left">GO:0048754</td>
<td headers="Dns-c  description" class="gt_row gt_left">branching morphogenesis of an epithelial tube</td>
<td headers="Dns-c  FDR" class="gt_row gt_left">&lt;.01</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Dns-d">Dns-d</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Dns-d  geneSet" class="gt_row gt_left">GO:0005794</td>
<td headers="Dns-d  description" class="gt_row gt_left">Golgi apparatus</td>
<td headers="Dns-d  FDR" class="gt_row gt_left">0.02</td></tr>
    <tr><td headers="Dns-d  geneSet" class="gt_row gt_left">GO:0001883</td>
<td headers="Dns-d  description" class="gt_row gt_left">purine nucleoside binding</td>
<td headers="Dns-d  FDR" class="gt_row gt_left">0.02</td></tr>
    <tr><td headers="Dns-d  geneSet" class="gt_row gt_left">GO:0045934</td>
<td headers="Dns-d  description" class="gt_row gt_left">negative regulation of nucleobase-containing compound metabolic process</td>
<td headers="Dns-d  FDR" class="gt_row gt_left">0.03</td></tr>
    <tr><td headers="Dns-d  geneSet" class="gt_row gt_left">GO:0001882</td>
<td headers="Dns-d  description" class="gt_row gt_left">nucleoside binding</td>
<td headers="Dns-d  FDR" class="gt_row gt_left">0.05</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="3" class="gt_group_heading" style="font-weight: bold;" scope="colgroup" id="Dns-e">Dns-e</th>
    </tr>
    <tr class="gt_row_group_first"><td headers="Dns-e  geneSet" class="gt_row gt_left">GO:0022613</td>
<td headers="Dns-e  description" class="gt_row gt_left">ribonucleoprotein complex biogenesis</td>
<td headers="Dns-e  FDR" class="gt_row gt_left">0.2</td></tr>
  </tbody>
  
  
</table>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.1 (2024-06-14)
Platform: aarch64-apple-darwin20
Running under: macOS Sonoma 14.7.1

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: America/New_York
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] gt_0.11.1       cowplot_1.1.3   lubridate_1.9.3 forcats_1.0.0  
 [5] stringr_1.5.1   dplyr_1.1.4     purrr_1.0.2     readr_2.1.5    
 [9] tidyr_1.3.1     tibble_3.2.1    ggplot2_3.5.1   tidyverse_2.0.0
[13] workflowr_1.7.1

loaded via a namespace (and not attached):
 [1] gtable_0.3.6        xfun_0.50           bslib_0.9.0        
 [4] htmlwidgets_1.6.4   ggrepel_0.9.6       processx_3.8.4     
 [7] lattice_0.22-6      callr_3.7.6         tzdb_0.4.0         
[10] quadprog_1.5-8      vctrs_0.6.5         tools_4.4.1        
[13] ps_1.8.0            generics_0.1.3      parallel_4.4.1     
[16] pkgconfig_2.0.3     Matrix_1.7-0        data.table_1.16.2  
[19] SQUAREM_2021.1      RcppParallel_5.1.9  truncnorm_1.0-9    
[22] lifecycle_1.0.4     farver_2.1.2        compiler_4.4.1     
[25] git2r_0.33.0        progress_1.2.3      munsell_0.5.1      
[28] RhpcBLASctl_0.23-42 getPass_0.2-4       httpuv_1.6.15      
[31] htmltools_0.5.8.1   sass_0.4.9          lazyeval_0.2.2     
[34] yaml_2.3.10         plotly_4.10.4       crayon_1.5.3       
[37] later_1.4.1         pillar_1.10.1       jquerylib_0.1.4    
[40] whisker_0.4.1       uwot_0.2.2          cachem_1.1.0       
[43] gtools_3.9.5        tidyselect_1.2.1    digest_0.6.37      
[46] Rtsne_0.17          stringi_1.8.4       ashr_2.2-63        
[49] labeling_0.4.3      rprojroot_2.0.4     fastmap_1.2.0      
[52] grid_4.4.1          invgamma_1.1        colorspace_2.1-1   
[55] cli_3.6.3           magrittr_2.0.3      withr_3.0.2        
[58] prettyunits_1.2.0   scales_1.3.0        promises_1.3.2     
[61] timechange_0.3.0    rmarkdown_2.29      httr_1.4.7         
[64] fastTopics_0.7-07   hms_1.1.3           pbapply_1.7-2      
[67] evaluate_1.0.3      knitr_1.49          viridisLite_0.4.2  
[70] irlba_2.3.5.1       rlang_1.1.5         mixsqp_0.3-54      
[73] Rcpp_1.0.14         glue_1.8.0          xml2_1.3.6         
[76] rstudioapi_0.17.0   jsonlite_1.8.9      R6_2.5.1           
[79] fs_1.6.5           </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
